# Беларускі пераклад anaconda.
# Copyright (C) 2004 Red Hat, Inc.
# This file is distributed under the same license as the anaconda package.
# Ales Nyakhaychyk <nab@mail.by>, 2004.
#
#
msgid ""
msgstr ""
"Project-Id-Version: anaconda HEAD\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2010-10-19 17:55-0700\n"
"PO-Revision-Date: 2004-09-26 18:13+0300\n"
"Last-Translator: Ales Nyakhaychyk <nab@mail.by>\n"
"Language-Team: Belorussian\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: anaconda:313
msgid "Press <enter> for a shell"
msgstr ""

#: anaconda:336
#, fuzzy
msgid "Not enough RAM"
msgstr "Бракуе прасторы"

#: anaconda:337
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""

#: anaconda:340
#, fuzzy
msgid " Starting text mode."
msgstr "Выкарыстоўваць тэкставы рэжым"

#: anaconda:364 pyanaconda/rescue.py:353 pyanaconda/rescue.py:379
#: pyanaconda/rescue.py:392 pyanaconda/rescue.py:475 pyanaconda/rescue.py:488
#: pyanaconda/rescue.py:499 pyanaconda/text.py:534 loader/windows.h:31
#: loader/cdinstall.c:229 loader/cdinstall.c:233 loader/cdinstall.c:249
#: loader/cdinstall.c:252 loader/cdinstall.c:428 loader/cdinstall.c:432
#: loader/cdinstall.c:505 loader/dirbrowser.c:129 loader/driverdisk.c:228
#: loader/driverdisk.c:408 loader/driverdisk.c:440 loader/driverdisk.c:471
#: loader/driverdisk.c:506 loader/driverdisk.c:522 loader/driverdisk.c:534
#: loader/driverdisk.c:542 loader/driverdisk.c:709 loader/driverdisk.c:748
#: loader/driverselect.c:79 loader/driverselect.c:179 loader/hdinstall.c:115
#: loader/hdinstall.c:266 loader/hdinstall.c:325 loader/hdinstall.c:360
#: loader/hdinstall.c:440 loader/hdinstall.c:483 loader/hdinstall.c:496
#: loader/kbd.c:119 loader/kickstart.c:135 loader/kickstart.c:145
#: loader/kickstart.c:187 loader/kickstart.c:192 loader/kickstart.c:299
#: loader/kickstart.c:339 loader/kickstart.c:355 loader/kickstart.c:382
#: loader/kickstart.c:571 loader/lang.c:114 loader/lang.c:373
#: loader/loader.c:433 loader/loader.c:469 loader/loader.c:509
#: loader/loader.c:527 loader/loader.c:544 loader/loader.c:581
#: loader/loader.c:1212 loader/loader.c:1399 loader/mediacheck.c:47
#: loader/mediacheck.c:86 loader/mediacheck.c:93 loader/mediacheck.c:97
#: loader/mediacheck.c:106 loader/method.c:286 loader/method.c:360
#: loader/net.c:621 loader/net.c:1001 loader/net.c:1654 loader/net.c:1675
#: loader/net.c:1981 loader/net.c:2000 loader/net.c:2012
#: loader/nfsinstall.c:116 loader/nfsinstall.c:309 loader/nfsinstall.c:326
#: loader/nfsinstall.c:414 loader/urlinstall.c:137 loader/urlinstall.c:159
#: loader/urlinstall.c:221 loader/urlinstall.c:374 loader/urlinstall.c:385
#: loader/urlinstall.c:392 loader/urls.c:269 loader/urls.c:340
#: loader/urls.c:346 loader/urls.c:355 pyanaconda/textw/constants_text.py:44
msgid "OK"
msgstr "Добра"

#: anaconda:727
#, fuzzy
msgid "Would you like to use VNC?"
msgstr "Які загрузчык вы жадаеце выкарыстоўваць?"

#: anaconda:728
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""

#: anaconda:754
#, fuzzy
msgid "Graphical installation is not available. Starting text mode."
msgstr "Графічная ўсталёўка недаступна... Запуск тэкставага рэжыму."

#: anaconda:762
msgid "DISPLAY variable not set. Starting text mode."
msgstr ""

#: pyanaconda/backend.py:135
#, fuzzy, python-format
msgid "Upgrading %s\n"
msgstr ""
"Абнаўленьне %s пакетаў\n"
"\n"

#: pyanaconda/backend.py:137
#, fuzzy, python-format
msgid "Installing %s\n"
msgstr "_Усталяваць %s"

#: pyanaconda/backend.py:150
msgid "Copying File"
msgstr "Капіяваньне файла"

#: pyanaconda/backend.py:151
msgid "Transferring install image to hard drive"
msgstr ""

#: pyanaconda/backend.py:161
#, fuzzy
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is often cause by damaged or low quality media."
msgstr ""
"Узьнікла памылка чытаньня ўсталёўкі з ISO вобразаў. Калі ласка, праверце "
"вашыя ISO вобразы й паўтарыце спробу."

#: pyanaconda/backend.py:165
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""

#: pyanaconda/backend.py:169 pyanaconda/image.py:286 pyanaconda/livecd.py:198
#: pyanaconda/livecd.py:440 pyanaconda/partIntfHelpers.py:233
#: pyanaconda/text.py:355 pyanaconda/text.py:359 pyanaconda/yuminstall.py:466
#: pyanaconda/yuminstall.py:838 pyanaconda/yuminstall.py:956
#: pyanaconda/yuminstall.py:961 pyanaconda/yuminstall.py:1225
#: pyanaconda/yuminstall.py:1285 pyanaconda/yuminstall.py:1476
#: pyanaconda/yuminstall.py:1498 pyanaconda/iw/advanced_storage.py:86
#: pyanaconda/iw/advanced_storage.py:96 pyanaconda/iw/advanced_storage.py:170
#: pyanaconda/iw/advanced_storage.py:173 pyanaconda/iw/advanced_storage.py:201
#: pyanaconda/iw/autopart_type.py:95 pyanaconda/iw/cleardisks_gui.py:44
#: pyanaconda/iw/cleardisks_gui.py:54 pyanaconda/iw/filter_gui.py:431
#: pyanaconda/iw/osbootwidget.py:211 pyanaconda/iw/osbootwidget.py:220
#: pyanaconda/iw/raid_dialog_gui.py:223 pyanaconda/iw/task_gui.py:54
#: pyanaconda/iw/task_gui.py:152 pyanaconda/iw/task_gui.py:305
#: pyanaconda/iw/task_gui.py:469 loader/windows.h:31 loader/cdinstall.c:229
#: loader/cdinstall.c:505 loader/driverdisk.c:228 loader/driverdisk.c:471
#: loader/driverdisk.c:506 loader/driverdisk.c:534 loader/driverdisk.c:542
#: loader/driverdisk.c:609 loader/hdinstall.c:115 loader/hdinstall.c:325
#: loader/hdinstall.c:440 loader/hdinstall.c:483 loader/hdinstall.c:496
#: loader/kickstart.c:299 loader/lang.c:114 loader/loader.c:433
#: loader/loader.c:544 loader/loader.c:1212 loader/mediacheck.c:47
#: loader/mediacheck.c:86 loader/mediacheck.c:93 loader/mediacheck.c:97
#: loader/method.c:286 loader/method.c:360 loader/nfsinstall.c:309
#: loader/nfsinstall.c:326 loader/urlinstall.c:137 loader/urlinstall.c:159
#: loader/urlinstall.c:221 loader/urls.c:340 loader/urls.c:346
#: loader/urls.c:355 pyanaconda/storage/__init__.py:230
#: pyanaconda/storage/__init__.py:1818 pyanaconda/storage/__init__.py:1905
#: pyanaconda/storage/zfcp.py:375 pyanaconda/textw/upgrade_text.py:181
#: pyanaconda/textw/upgrade_text.py:188
msgid "Error"
msgstr "Памылка"

#: pyanaconda/bootloader.py:55 pyanaconda/bootloader.py:221
#: pyanaconda/bootloader.py:227 pyanaconda/gui.py:1081 pyanaconda/image.py:80
#: pyanaconda/installinterfacebase.py:51 pyanaconda/installinterfacebase.py:82
#: pyanaconda/installinterfacebase.py:142 pyanaconda/packages.py:314
#: pyanaconda/text.py:411 pyanaconda/yuminstall.py:1454
#: pyanaconda/yuminstall.py:1662 pyanaconda/yuminstall.py:1697
#: pyanaconda/iw/blpasswidget.py:148 pyanaconda/iw/upgrade_swap_gui.py:190
#: pyanaconda/iw/upgrade_swap_gui.py:198 pyanaconda/iw/upgrade_swap_gui.py:205
#: pyanaconda/textw/upgrade_text.py:193
msgid "Warning"
msgstr "Увага"

#: pyanaconda/bootloader.py:56
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""

#: pyanaconda/bootloader.py:60 pyanaconda/gui.py:1104 pyanaconda/gui.py:1141
#: pyanaconda/gui.py:1342 pyanaconda/image.py:89 pyanaconda/kickstart.py:156
#: pyanaconda/livecd.py:205 pyanaconda/livecd.py:447
#: pyanaconda/packages.py:131 pyanaconda/upgrade.py:58
#: pyanaconda/upgrade.py:98 pyanaconda/upgrade.py:254
#: pyanaconda/yuminstall.py:257 pyanaconda/yuminstall.py:756
#: pyanaconda/yuminstall.py:958 pyanaconda/yuminstall.py:963
#: pyanaconda/yuminstall.py:1043 pyanaconda/yuminstall.py:1049
#: pyanaconda/yuminstall.py:1204 pyanaconda/yuminstall.py:1221
#: pyanaconda/yuminstall.py:1275 pyanaconda/yuminstall.py:1463
#: pyanaconda/yuminstall.py:1482 pyanaconda/yuminstall.py:1505
#: pyanaconda/storage/__init__.py:114 pyanaconda/storage/__init__.py:123
#: pyanaconda/storage/__init__.py:234 pyanaconda/storage/__init__.py:1999
#, fuzzy
msgid "_Exit installer"
msgstr "Рэдагаваць карыстальніка"

#: pyanaconda/bootloader.py:60 pyanaconda/image.py:90 pyanaconda/image.py:249
#: pyanaconda/kickstart.py:1383 pyanaconda/kickstart.py:1422
#: pyanaconda/upgrade.py:58 pyanaconda/upgrade.py:98
#: pyanaconda/yuminstall.py:1280 pyanaconda/yuminstall.py:1464
#: pyanaconda/iw/partition_gui.py:1599 pyanaconda/storage/__init__.py:2000
msgid "_Continue"
msgstr "П_рацяг"

#: pyanaconda/bootloader.py:158
msgid "Bootloader"
msgstr "Загрузчык"

#: pyanaconda/bootloader.py:158
#, fuzzy
msgid "Installing bootloader."
msgstr "Усталёўваньне загрузчыка..."

#: pyanaconda/bootloader.py:222
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""

#: pyanaconda/bootloader.py:228
#, fuzzy
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""
"У сыстэме адсутнічаюць усталяваныя пакеты ядра. Наладка загрузчыка ня будзе "
"зьменена."

#: pyanaconda/cmdline.py:56
msgid "Completed"
msgstr "Завершана"

#: pyanaconda/cmdline.py:64
#, fuzzy
msgid "In progress"
msgstr "Ідзе праца...   "

#: pyanaconda/cmdline.py:89 pyanaconda/gui.py:1136
#: pyanaconda/kickstart.py:1167 pyanaconda/kickstart.py:1247
#: pyanaconda/kickstart.py:1255 pyanaconda/kickstart.py:1293
#: pyanaconda/kickstart.py:1301 pyanaconda/text.py:392
#, fuzzy, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"Наступныя памылкі былі знойдзены цягам разбору наладкі kickstart:\n"
"\n"
"%s"

#: pyanaconda/cmdline.py:101
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""

#: pyanaconda/cmdline.py:120 pyanaconda/cmdline.py:127
#: pyanaconda/cmdline.py:134 pyanaconda/cmdline.py:141
msgid "Can't have a question in command line mode!"
msgstr "Немагчыма запытацца ў рэжыме загаднага разка!"

#: pyanaconda/constants.py:74
#, fuzzy
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""
"Адбылася невыправімая памылка. Хутчэй заўсё гэта памылка ў праграмным "
"забесьпячэньні. Калі ласка, стварэце копію ўсяго тэкста памылкі; запоўніце "
"падрабязнае паведамленьне аб памылцы ў Анакондзе на http://bugzilla.redhat."
"com/bugzilla/"

#: pyanaconda/constants.py:80
msgid " with the provider of this software."
msgstr ""

#: pyanaconda/constants.py:84
#, python-format
msgid " against anaconda at %s"
msgstr ""

#: pyanaconda/gui.py:108
#, fuzzy
msgid "An error occurred saving screenshots to disk."
msgstr "Адбылася памылка пад час капіяваньня здымкаў экрана."

#: pyanaconda/gui.py:125
msgid "Screenshots Copied"
msgstr "Здымкі экрана скапіяваны"

#: pyanaconda/gui.py:126
#, fuzzy
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"Здымкі экрана былі захаваны ў гэтую тэчку:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"Вы можаце атрымаць доступ да іх, калі пасьля\n"
"перазагружкі зайдзеце як root."

#: pyanaconda/gui.py:165
msgid "Saving Screenshot"
msgstr "Захаваньне здымка экрана"

#: pyanaconda/gui.py:166
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "Здымак экрана захаваны пад назвай %s."

#: pyanaconda/gui.py:169
msgid "Error Saving Screenshot"
msgstr "Памылка захаваньня здымка экрана"

#: pyanaconda/gui.py:170
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""

#: pyanaconda/gui.py:525 pyanaconda/text.py:150
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""

#: pyanaconda/gui.py:544 pyanaconda/gui.py:552 pyanaconda/text.py:188
#: pyanaconda/text.py:198
#, fuzzy
msgid "Error with passphrase"
msgstr "Памылка з паролем"

#: pyanaconda/gui.py:545 pyanaconda/text.py:189
#, fuzzy
msgid "The passphrases you entered were different.  Please try again."
msgstr "Паролі, якія вы ўвялі не супадаюць. Калі ласка, паўтарыце спробу."

#: pyanaconda/gui.py:553
#, fuzzy
msgid "The passphrase must be at least eight characters long."
msgstr "Пароль мусіць быць даўжэйшым за пяць знакаў."

#: pyanaconda/gui.py:587 pyanaconda/text.py:226
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/gui.py:678 pyanaconda/gui.py:1342
#: pyanaconda/partIntfHelpers.py:153 pyanaconda/partIntfHelpers.py:338
#: pyanaconda/text.py:101 pyanaconda/text.py:102
#: pyanaconda/iw/account_gui.py:131 loader/dirbrowser.c:129
#: loader/driverdisk.c:409 loader/kickstart.c:382 loader/loader.c:469
#: loader/loader.c:581 pyanaconda/textw/constants_text.py:48
msgid "Cancel"
msgstr "Скасаваць"

#: pyanaconda/gui.py:690 pyanaconda/gui.py:691 pyanaconda/gui.py:809
#: pyanaconda/gui.py:810 data/ui/anaconda.glade.h:3
msgid "_Debug"
msgstr "А_дладка"

#: pyanaconda/gui.py:933
msgid "Wireless setup"
msgstr ""

#: pyanaconda/gui.py:934
msgid "Scanning access points for wireless devices"
msgstr ""

#: pyanaconda/gui.py:960 pyanaconda/textw/netconfig_text.py:399
msgid "Waiting for NetworkManager"
msgstr ""

#: pyanaconda/gui.py:961
#, python-format
msgid "Waiting for NetworkManager to activate these devices: %s"
msgstr ""

#: pyanaconda/gui.py:982
#, python-format
msgid "Failed to activate these network interfaces: %s"
msgstr ""

#: pyanaconda/gui.py:985
#, fuzzy
msgid "Network Configuration"
msgstr "Наладка VNC"

#: pyanaconda/gui.py:994 pyanaconda/iw/netconfig_dialog.py:213
#, fuzzy, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr "Узьнікла памылка пад час пошуку пакетаў для абнаўленьня."

#: pyanaconda/gui.py:996
#, fuzzy
msgid "Error Enabling Network"
msgstr "Наладка сеткі для %s"

#: pyanaconda/gui.py:1082 pyanaconda/text.py:412
msgid ""
"You do not have an active network connection.  This is required by some "
"exception saving methods.  Would you like to configure your network now?"
msgstr ""

#: pyanaconda/gui.py:1088 pyanaconda/kickstart.py:150 pyanaconda/rescue.py:262
#: pyanaconda/text.py:418 pyanaconda/yuminstall.py:751
#: pyanaconda/yuminstall.py:1199 pyanaconda/iw/task_gui.py:311
#, fuzzy
msgid "No Network Available"
msgstr "Сеткавыя прылады"

#: pyanaconda/gui.py:1089 pyanaconda/text.py:419
msgid "Remote exception saving methods will not work."
msgstr ""

#: pyanaconda/gui.py:1138 pyanaconda/text.py:394
msgid "Error Parsing Kickstart Config"
msgstr "Памылка разбору наладкі Kickstart"

#: pyanaconda/gui.py:1166
msgid "default:LTR"
msgstr "default:LTR"

#: pyanaconda/gui.py:1246 pyanaconda/text.py:511
msgid "Error!"
msgstr "Памылка!"

#: pyanaconda/gui.py:1247 pyanaconda/text.py:512
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""

#: pyanaconda/gui.py:1252 pyanaconda/image.py:163 pyanaconda/image.py:192
#: pyanaconda/packages.py:331 pyanaconda/storage/__init__.py:1817
msgid "_Exit"
msgstr "Вы_йсьці"

#: pyanaconda/gui.py:1253 pyanaconda/image.py:163 pyanaconda/image.py:192
#: pyanaconda/livecd.py:205 pyanaconda/yuminstall.py:835
#: pyanaconda/yuminstall.py:1221 pyanaconda/yuminstall.py:1275
#: pyanaconda/yuminstall.py:1482
msgid "_Retry"
msgstr "_Паўтарыць"

#: pyanaconda/gui.py:1255 pyanaconda/storage/partitioning.py:283
#, fuzzy
msgid "The system will now reboot."
msgstr "Зараз сыстэма будзе перазагружана."

#: pyanaconda/gui.py:1256 pyanaconda/image.py:249 pyanaconda/packages.py:335
#: pyanaconda/yuminstall.py:1523
msgid "_Reboot"
msgstr "Перазагруз_іць"

#: pyanaconda/gui.py:1258
#, fuzzy
msgid "Exiting"
msgstr "Выйсьці"

#: pyanaconda/gui.py:1339 pyanaconda/livecd.py:126 pyanaconda/text.py:271
#: pyanaconda/upgrade.py:245
#, fuzzy
msgid "Exit installer"
msgstr "Рэдагаваць карыстальніка"

#: pyanaconda/gui.py:1340
#, fuzzy
msgid "Are you sure you wish to exit the installer?"
msgstr "На якой мовы вы жадаеце весьці ўсталёўку?"

#: pyanaconda/gui.py:1349
#, python-format
msgid "%s Installer"
msgstr ""

#: pyanaconda/gui.py:1355
msgid "Unable to load title bar"
msgstr "Немагчыма загрузіць загаловак вакна"

#: pyanaconda/gui.py:1433
msgid "Install Window"
msgstr "Вакно ўсталёўкі"

#: pyanaconda/image.py:81
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""

#: pyanaconda/image.py:155
msgid "Couldn't Mount ISO Source"
msgstr ""

#: pyanaconda/image.py:156
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""

#: pyanaconda/image.py:183
msgid "Missing ISO 9660 Image"
msgstr ""

#: pyanaconda/image.py:184
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""

#: pyanaconda/image.py:239
msgid "Required Install Media"
msgstr "Патрэбны носьбіт усталёўкі"

#: pyanaconda/image.py:240
#, python-format
msgid ""
"The software you have selected to install will require the following "
"%(productName)s %(productVersion)s discs:\n"
"\n"
"%(reqcdstr)s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""

#: pyanaconda/image.py:249 pyanaconda/livecd.py:446 pyanaconda/packages.py:335
#: pyanaconda/upgrade.py:253 pyanaconda/yuminstall.py:963
#: pyanaconda/yuminstall.py:1049 pyanaconda/yuminstall.py:1463
#: pyanaconda/yuminstall.py:1505 pyanaconda/yuminstall.py:1523
#: data/ui/anaconda.glade.h:2
msgid "_Back"
msgstr "На_зад"

#: pyanaconda/image.py:287
#, fuzzy, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"Пад час стварэньня загрузачнага дыска ўзьнікла памылка. Калі ласка, "
"упэўніцеся, што гнуткі дыск прысутнічае ў першым дыскаводзе."

#: pyanaconda/installclass.py:67
msgid "Install on System"
msgstr ""

#: pyanaconda/installinterfacebase.py:52
#, python-format
msgid ""
"Disk %s contains BIOS RAID metadata, but is not part of any recognized BIOS "
"RAID sets. Ignoring disk %s."
msgid_plural ""
"Disks %s contain BIOS RAID metadata, but are not part of any recognized BIOS "
"RAID sets. Ignoring disks %s."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/installinterfacebase.py:83
#, python-format
msgid ""
"Error processing drive:\n"
"\n"
"%(path)s\n"
"%(size)-0.fMB\n"
"%(description)s\n"
"\n"
"This device may need to be reinitialized.\n"
"\n"
"REINITIALIZING WILL CAUSE ALL DATA TO BE LOST!\n"
"\n"
"This action may also be applied to all other disks needing reinitialization."
"%(details)s"
msgstr ""

#: pyanaconda/installinterfacebase.py:92
#: pyanaconda/installinterfacebase.py:150
#: pyanaconda/installinterfacebase.py:184
#, fuzzy
msgid "_Ignore"
msgstr "Ігнараваць"

#: pyanaconda/installinterfacebase.py:93
#: pyanaconda/installinterfacebase.py:151
#, fuzzy
msgid "Ignore _all"
msgstr "Ігнараваць"

#: pyanaconda/installinterfacebase.py:94
#: pyanaconda/installinterfacebase.py:152
msgid "_Re-initialize"
msgstr ""

#: pyanaconda/installinterfacebase.py:95
#: pyanaconda/installinterfacebase.py:153
msgid "Re-ini_tialize all"
msgstr ""

#: pyanaconda/installinterfacebase.py:143
#, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %(msg)s.  You can reinitialize all related "
"PVs (%(pvs)s) which will erase the LVM metadata, or ignore which will "
"preserve the contents.  This action may also be applied to all other PVs "
"with inconsistent metadata."
msgstr ""

#: pyanaconda/installinterfacebase.py:171
msgid "Unformatted DASD Device Found"
msgid_plural "Unformatted DASD Devices Found"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/installinterfacebase.py:173
#, python-format
msgid ""
"Format uninitialized DASD device?\n"
"\n"
"There is %d uninitialized DASD device on this system.  To continue "
"installation, the device must be formatted.  Formatting will remove any data "
"on this device."
msgid_plural ""
"Format uninitialized DASD devices?\n"
"\n"
"There are %d uninitialized DASD devices on this system.  To continue "
"installation, the devices must be formatted.  Formatting will remove any "
"data on these devices."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/installinterfacebase.py:184 pyanaconda/iw/partition_gui.py:898
msgid "_Format"
msgstr ""

#: pyanaconda/iutil.py:867
#, python-format
msgid ""
"Error: On open, cannot set reIPL method to %(reipl_type)s (%(filename)s: "
"%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:878
#, python-format
msgid ""
"Error: On write, cannot set reIPL method to %(reipl_type)s (%(filename)s: "
"%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:887
#, python-format
msgid ""
"Error: On close, cannot set reIPL method to %(reipl_type)s (%(filename)s: "
"%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:906
#, python-format
msgid "Error: Could not set %(device)s as reIPL device (%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:916
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr ""

#: pyanaconda/iutil.py:925
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr ""

#: pyanaconda/iutil.py:938
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from DASD device %s to continue "
"installation"
msgstr ""

#: pyanaconda/iutil.py:960
#, python-format
msgid "Error: reading FCP property %(syspath_property)s for reIPL (%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:975
#, python-format
msgid "Error: writing FCP property %(reipl_property)s for reIPL (%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:990
#, python-format
msgid ""
"Error: writing default FCP property %(reipl_property)s for reIPL (%(e)s)"
msgstr ""

#: pyanaconda/iutil.py:1005
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from FCP %(device)s with WWPN "
"%(wwpn)s and LUN %(lun)s to continue installation"
msgstr ""

#: pyanaconda/iutil.py:1022
msgid ""
"After shutdown, please perform a manual IPL from the device now containing /"
"boot to continue installation"
msgstr ""

#: pyanaconda/iutil.py:1033
msgid "Error determining boot device's disk name"
msgstr ""

#: pyanaconda/iutil.py:1037
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr ""

#: pyanaconda/kickstart.py:115
#, fuzzy, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  You may "
"examine the output in %(msgs)s.  This is a fatal error and installation will "
"be aborted.  Press the OK button to exit the installer."
msgstr ""
"Узьнікла памылка пад час ініцыялізаваньня прасторы падмену %s. Гэта "
"сур'ёзная праблема й усталёўка ня можа працягвацца далей.\n"
"\n"
"Націсніце <Enter> для перазагрузкі сыстэмы."

#: pyanaconda/kickstart.py:123 pyanaconda/kickstart.py:125
msgid "Scriptlet Failure"
msgstr ""

#: pyanaconda/kickstart.py:151
msgid ""
"Encryption key escrow requires networking, but there was an error enabling "
"the network on your system."
msgstr ""

#: pyanaconda/kickstart.py:165
#, fuzzy, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"Наступныя памылкі былі знойдзены цягам разбору наладкі kickstart:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:1266
#, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr ""

#: pyanaconda/kickstart.py:1269
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr ""

#: pyanaconda/kickstart.py:1324 pyanaconda/livecd.py:228
#, fuzzy
msgid "Post-Installation"
msgstr "%s %s усталёўка"

#: pyanaconda/kickstart.py:1325
#, fuzzy
msgid "Running post-installation scripts"
msgstr "%s %s усталёўка"

#: pyanaconda/kickstart.py:1341
#, fuzzy
msgid "Pre-Installation"
msgstr "Усталёўваньне пакета"

#: pyanaconda/kickstart.py:1342
#, fuzzy
msgid "Running pre-installation scripts"
msgstr "%s %s усталёўка"

#: pyanaconda/kickstart.py:1374
msgid "Missing Package"
msgstr "Адсутнічае пакет"

#: pyanaconda/kickstart.py:1375
#, fuzzy, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""
"Вы пазначылі што пакет \"%s\" павінны быць усталяваны. Гэты пакет не йснуе. "
"Выжадаеце працягваць ці абарваць усталёўку?"

#: pyanaconda/kickstart.py:1381 pyanaconda/kickstart.py:1420
msgid "_Abort"
msgstr "А_барваць"

#: pyanaconda/kickstart.py:1382 pyanaconda/kickstart.py:1421
#, fuzzy
msgid "_Ignore All"
msgstr "Ігнараваць"

#: pyanaconda/kickstart.py:1412
msgid "Missing Group"
msgstr "Адсутнічае група"

#: pyanaconda/kickstart.py:1413
#, fuzzy, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""
"Вы пазначылі што група \"%s\" павінна быць усталявана. Гэтая пакет не йснуе. "
"Выжадаеце працягваць ці абарваць усталёўку?"

#: pyanaconda/kickstart.py:1511
#, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""

#: pyanaconda/livecd.py:121
#, fuzzy
msgid "Unable to find image"
msgstr "Немагчыма адшукаць вобраз усталёўкі %s"

#: pyanaconda/livecd.py:122
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""

#: pyanaconda/livecd.py:180
msgid "Copying live image to hard drive."
msgstr ""

#: pyanaconda/livecd.py:199
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""

#: pyanaconda/livecd.py:229
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""

#: pyanaconda/livecd.py:441
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""

#: pyanaconda/network.py:63
#, fuzzy
msgid "Hostname must be 255 or fewer characters in length."
msgstr "Назва вузла павінна быць ня большай за 64 знака."

#: pyanaconda/network.py:69
#, fuzzy
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
"'0-9'"
msgstr "Назва вузла павінна начынацца зь літар \"a-z\" альбо \"A-Z\""

#: pyanaconda/network.py:74
#, fuzzy
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr ""
"Назва вузла можу ўтрымліваць толькі знакі \"a-z\",·\"A-Z\",·\"-\" альбо \".\""

#: pyanaconda/network.py:118
#, fuzzy
msgid "IP address is missing."
msgstr "Прапушчаны IP адрас"

#: pyanaconda/network.py:122
#, fuzzy
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr "IP адрас мусіць утрымліваць ліжбы ў прамежку ад 0 да 255"

#: pyanaconda/network.py:125
#, fuzzy, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "%s - нерэчаісная назва вузла.ы"

#: pyanaconda/network.py:127
#, fuzzy, python-format
msgid "'%s' is an invalid IP address."
msgstr "%s - нерэчаісная назва вузла.ы"

#: pyanaconda/packages.py:112
#, fuzzy
msgid "Resizing Failed"
msgstr "Выцягваньне"

#: pyanaconda/packages.py:113
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr ""

#: pyanaconda/packages.py:121
#, fuzzy
msgid "Migration Failed"
msgstr "Выцягваньне"

#: pyanaconda/packages.py:122
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr ""

#: pyanaconda/packages.py:131
msgid "_File Bug"
msgstr ""

#: pyanaconda/packages.py:315
#, python-format
msgid ""
"Warning!  This is pre-release software!\n"
"\n"
"Thank you for downloading this pre-release of %(productName)s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %(bugzillaUrl)s\n"
"\n"
"and file a report against '%(fileagainst)s'.\n"
msgstr ""

#: pyanaconda/packages.py:331
#, fuzzy
msgid "_Install Anyway"
msgstr "_Усталяваць %s"

#: pyanaconda/packages.py:334
msgid "Your system will now be rebooted..."
msgstr "Сыстэма зараз будзе перазагружана..."

#: pyanaconda/packages.py:336
msgid "Warning! This is pre-release software!"
msgstr ""

#: pyanaconda/partIntfHelpers.py:41
msgid "Please enter a volume group name."
msgstr ""

#: pyanaconda/partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr ""

#: pyanaconda/partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr ""

#: pyanaconda/partIntfHelpers.py:53
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: pyanaconda/partIntfHelpers.py:63
msgid "Please enter a logical volume name."
msgstr ""

#: pyanaconda/partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr ""

#: pyanaconda/partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr ""

#: pyanaconda/partIntfHelpers.py:77
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: pyanaconda/partIntfHelpers.py:105
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""

#: pyanaconda/partIntfHelpers.py:117 pyanaconda/partIntfHelpers.py:124
msgid "Unable To Delete"
msgstr ""

#: pyanaconda/partIntfHelpers.py:118
msgid "You must first select a partition to delete."
msgstr ""

#: pyanaconda/partIntfHelpers.py:149 pyanaconda/partIntfHelpers.py:337
#: pyanaconda/iw/lvm_dialog_gui.py:877
msgid "Confirm Delete"
msgstr "Пацьверджаньне выдаленьня"

#: pyanaconda/partIntfHelpers.py:150
#, fuzzy, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "Вы зьбіраецесь выдаліць усе падзелы на прыладе \"/dev/%s\"."

#: pyanaconda/partIntfHelpers.py:153 pyanaconda/partIntfHelpers.py:338
#: pyanaconda/iw/lvm_dialog_gui.py:880 pyanaconda/iw/lvm_dialog_gui.py:1450
#: pyanaconda/iw/osbootwidget.py:104 pyanaconda/iw/partition_gui.py:1812
#: pyanaconda/iw/partition_gui.py:1824
msgid "_Delete"
msgstr "В_ыдаліць"

#: pyanaconda/partIntfHelpers.py:200
msgid "Notice"
msgstr ""

#: pyanaconda/partIntfHelpers.py:201
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/partIntfHelpers.py:216
msgid "Format as Swap?"
msgstr "Фарматаваць як Swap?"

#: pyanaconda/partIntfHelpers.py:217
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:232
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""

#: pyanaconda/partIntfHelpers.py:237
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""

#: pyanaconda/partIntfHelpers.py:245
msgid "Format?"
msgstr "Фарматаваць"

#: pyanaconda/partIntfHelpers.py:245 pyanaconda/iw/partition_gui.py:1597
msgid "_Modify Partition"
msgstr "Зь_мяніць падзел"

#: pyanaconda/partIntfHelpers.py:245
msgid "Do _Not Format"
msgstr "_Не фарматаваць"

#: pyanaconda/partIntfHelpers.py:253
msgid "Error with Partitioning"
msgstr ""

#: pyanaconda/partIntfHelpers.py:254
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of "
"%(productName)s.\n"
"\n"
"%(errorstr)s"
msgstr ""

#: pyanaconda/partIntfHelpers.py:270
msgid "Partitioning Warning"
msgstr ""

#: pyanaconda/partIntfHelpers.py:271
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:285
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""

#: pyanaconda/partIntfHelpers.py:288
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""

#: pyanaconda/partIntfHelpers.py:294
msgid "Format Warning"
msgstr ""

#: pyanaconda/partIntfHelpers.py:321
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""

#: pyanaconda/partIntfHelpers.py:325
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr ""

#: pyanaconda/partIntfHelpers.py:328
msgid "You are about to delete a RAID device."
msgstr ""

#: pyanaconda/partIntfHelpers.py:330
#, fuzzy, python-format
msgid "You are about to delete the %s partition."
msgstr "Вы зьбіраецесь выдаліць усе падзелы на прыладе \"/dev/%s\"."

#: pyanaconda/partIntfHelpers.py:334
#, fuzzy, python-format
msgid "You are about to delete the %(type)s %(name)s"
msgstr "Вы зьбіраецесь выдаліць усе падзелы на прыладе \"/dev/%s\"."

#: pyanaconda/partIntfHelpers.py:345
msgid "Confirm Reset"
msgstr ""

#: pyanaconda/partIntfHelpers.py:346
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""

#: pyanaconda/platform.py:98 pyanaconda/platform.py:344
#: pyanaconda/platform.py:412 pyanaconda/platform.py:520
#: data/ui/create-storage.glade.h:15
msgid "RAID Device"
msgstr "RAID прылада"

#: pyanaconda/platform.py:99 pyanaconda/platform.py:102
#: pyanaconda/platform.py:345 pyanaconda/platform.py:413
#: pyanaconda/platform.py:521 pyanaconda/platform.py:524
msgid "Master Boot Record (MBR)"
msgstr "Галоўны загрузачны запіс (MBR)"

#: pyanaconda/platform.py:101 pyanaconda/platform.py:523
msgid "First sector of boot partition"
msgstr "Першы сэктар загрузачнага падзелу"

#: pyanaconda/platform.py:113
#, fuzzy
msgid "You have not created a bootable partition."
msgstr "Вы мусіце вылучыць падзел для рэдагаваньня"

#: pyanaconda/platform.py:118
msgid "Bootable partitions cannot be on a RAID device."
msgstr ""

#: pyanaconda/platform.py:120
msgid "Bootable partitions can only be on RAID1 devices."
msgstr ""

#: pyanaconda/platform.py:124
msgid "Bootable RAID1 set members must be partitions."
msgstr ""

#: pyanaconda/platform.py:129
msgid "Bootable partitions cannot be on a logical volume."
msgstr ""

#: pyanaconda/platform.py:136
#, fuzzy, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr "Выдліць усе падзелы з гэтае сыстэмы"

#: pyanaconda/platform.py:140 pyanaconda/platform.py:145
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""

#: pyanaconda/platform.py:217
#, fuzzy
msgid "EFI System Partition"
msgstr "Рэдагаваньне падзела"

#: pyanaconda/platform.py:229
#, fuzzy
msgid "You have not created a /boot/efi partition."
msgstr "Вы мусіце вылучыць падзел для рэдагаваньня"

#: pyanaconda/platform.py:235
msgid "/boot/efi is not EFI."
msgstr ""

#: pyanaconda/platform.py:251
#, python-format
msgid "%s must have a %s disk label."
msgstr ""

#: pyanaconda/platform.py:289
#, python-format
msgid "%s must have a bsd disk label."
msgstr ""

#: pyanaconda/platform.py:301
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr ""

#: pyanaconda/platform.py:347
msgid "PPC PReP Boot"
msgstr "PPC PReP Boot"

#: pyanaconda/platform.py:363
msgid "The boot partition must be within the first 4MB of the disk."
msgstr ""

#: pyanaconda/platform.py:415 pyanaconda/platform.py:418
msgid "Apple Bootstrap"
msgstr "Apple Bootstrap"

#: pyanaconda/platform.py:432
#, python-format
msgid "%s must have a mac disk label."
msgstr ""

#: pyanaconda/rescue.py:221
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: pyanaconda/rescue.py:235
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: pyanaconda/rescue.py:254
msgid "Setup Networking"
msgstr ""

#: pyanaconda/rescue.py:255
msgid "Do you want to start the network interfaces on this system?"
msgstr ""

#: pyanaconda/rescue.py:256 loader/driverdisk.c:649 loader/driverdisk.c:659
#: loader/hdinstall.c:212 pyanaconda/textw/constants_text.py:56
msgid "Yes"
msgstr "Так"

#: pyanaconda/rescue.py:256 pyanaconda/rescue.py:258 loader/driverdisk.c:649
#: loader/driverdisk.c:659 pyanaconda/textw/constants_text.py:60
msgid "No"
msgstr "Не"

#: pyanaconda/rescue.py:263
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:374 pyanaconda/rescue.py:385
#: pyanaconda/rescue.py:470 pyanaconda/rescue.py:499
msgid "Rescue"
msgstr "Выратаваньне"

#: pyanaconda/rescue.py:300
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  If you need to activate SAN devices choose "
"'Advanced'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:311 pyanaconda/iw/partition_gui.py:785
#: loader/cdinstall.c:207 loader/cdinstall.c:215 loader/driverdisk.c:610
#: pyanaconda/storage/__init__.py:169 pyanaconda/storage/devicetree.py:90
msgid "Continue"
msgstr "Працяг"

#: pyanaconda/rescue.py:311 pyanaconda/rescue.py:320
msgid "Read-Only"
msgstr "Толькі чытаньне"

#: pyanaconda/rescue.py:311 pyanaconda/rescue.py:313 loader/cdinstall.c:249
#: loader/cdinstall.c:252 loader/method.c:333
#: pyanaconda/storage/__init__.py:1817 pyanaconda/textw/upgrade_text.py:139
msgid "Skip"
msgstr "Абмінуць"

#: pyanaconda/rescue.py:311 pyanaconda/rescue.py:316
msgid "Advanced"
msgstr ""

#: pyanaconda/rescue.py:350
msgid "System to Rescue"
msgstr "Сыстэма для выратаваньня"

#: pyanaconda/rescue.py:351
#, fuzzy
msgid "Which device holds the root partition of your installation?"
msgstr "Які падзел зьяўляецца каранёвым для вашае сыстэмы?"

#: pyanaconda/rescue.py:353 pyanaconda/rescue.py:357 pyanaconda/text.py:516
#: pyanaconda/text.py:518
msgid "Exit"
msgstr "Выйсьці"

#: pyanaconda/rescue.py:375
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:386
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:471
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:481
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: pyanaconda/rescue.py:484
msgid "Rescue Mode"
msgstr "Выратавальны рэжым"

#: pyanaconda/rescue.py:485
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:498
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: pyanaconda/text.py:155
msgid "Passphrase for encrypted device"
msgstr ""

#: pyanaconda/text.py:168
msgid "Also add this passphrase to all existing encrypted devices"
msgstr ""

#: pyanaconda/text.py:199
#, fuzzy, python-format
msgid "The passphrase must be at least %d character long."
msgid_plural "The passphrase must be at least %d characters long."
msgstr[0] "Пароль мусіць быць даўжэйшым за пяць знакаў."
msgstr[1] "Пароль мусіць быць даўжэйшым за пяць знакаў."

#: pyanaconda/text.py:233 data/ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr ""

#: pyanaconda/text.py:241 data/ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr ""

#: pyanaconda/text.py:356 pyanaconda/text.py:360
#, fuzzy
msgid "Repository editing is not available in text mode."
msgstr "Графічная ўсталёўка недаступна... Запуск тэкставага рэжыму."

#: pyanaconda/text.py:437
#, fuzzy, python-format
msgid "Welcome to %(productName)s for %(productArch)s"
msgstr "Вітаем у %s"

#: pyanaconda/text.py:439
#, python-format
msgid "Welcome to %s"
msgstr "Вітаем у %s"

#: pyanaconda/text.py:441
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""

#: pyanaconda/text.py:516 loader/net.c:110 loader/net.c:513 loader/net.c:568
#: loader/net.c:710 loader/net.c:718 loader/net.c:1148 loader/net.c:1156
msgid "Retry"
msgstr "Паўтарыць"

#: pyanaconda/text.py:530
msgid "Cancelled"
msgstr ""

#: pyanaconda/text.py:531
msgid "I can't go to the previous step from here. You will have to try again."
msgstr ""

#: pyanaconda/upgrade.py:51
msgid "Proceed with upgrade?"
msgstr "Зрабіць абнаўленьне?"

#: pyanaconda/upgrade.py:52
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"Файлавая сыстэма ўсталёўкі Linux, якую вы абралі для абнаўленьня ўжо "
"падмацавана. Вы ня зможаце вярнуцца назад пасьля гэтага моманту. n\"\n"

#: pyanaconda/upgrade.py:56
msgid "Would you like to continue with the upgrade?"
msgstr "Вы насамрэч жадаеце зрабіць абнаўленьне?"

#: pyanaconda/upgrade.py:91
#, fuzzy
msgid "Cannot Upgrade"
msgstr "Абнаўленьне сыстэмы"

#: pyanaconda/upgrade.py:92
#, python-format
msgid ""
"Your current installation cannot be upgraded. This is likely due to it being "
"too old. Only the previous two release may be upgraded. To upgrade older "
"releases you must first upgrade through all intermediate releases.\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/upgrade.py:232
msgid "Mount failed"
msgstr ""

#: pyanaconda/upgrade.py:233
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""

#: pyanaconda/upgrade.py:241 pyanaconda/upgrade.py:248
msgid "Upgrade root not found"
msgstr ""

#: pyanaconda/upgrade.py:242
msgid "The root for the previously installed system was not found."
msgstr ""

#: pyanaconda/upgrade.py:249
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""

#: pyanaconda/upgrade.py:272
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""

#: pyanaconda/upgrade.py:278
msgid "Absolute Symlinks"
msgstr ""

#: pyanaconda/upgrade.py:289
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""

#: pyanaconda/upgrade.py:295
msgid "Invalid Directories"
msgstr "Нерэчаісныя тэчкі"

#: pyanaconda/vnc.py:140
#, fuzzy, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "%s %s усталёўка на вузел %s"

#: pyanaconda/vnc.py:146
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: pyanaconda/vnc.py:175
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "Спроба далучыцца да vnc кліента на вузьле %s..."

#: pyanaconda/vnc.py:189
msgid "Connected!"
msgstr "Далучэньне ўсталявана!"

#: pyanaconda/vnc.py:192
msgid "Will try to connect again in 15 seconds..."
msgstr "Яшчэ адна спроба далучыцца адбудзецца праз 15 сэкунд..."

#: pyanaconda/vnc.py:198
#, fuzzy, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "Спробы далучыцца скончваюцца пасьля 50 разоў!\n"
msgstr[1] "Спробы далучыцца скончваюцца пасьля 50 разоў!\n"

#: pyanaconda/vnc.py:209
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"Калі ласка, самастойна далучыце вашага vnc кліента да %s каб пачаць "
"усталёўку."

#: pyanaconda/vnc.py:211
msgid "Please manually connect your vnc client to begin the install."
msgstr ""
"Калі ласка, самастойна далучыце вашага vnc кліента каб пачаць усталёўку."

#: pyanaconda/vnc.py:214
msgid "Starting VNC..."
msgstr "Запуск VNC..."

#: pyanaconda/vnc.py:239
msgid "The VNC server is now running."
msgstr "Паслужнік VNC працуе."

#: pyanaconda/vnc.py:252
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:257
#, fuzzy
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"УВАГА!!! Паслужнік VNC запушчаны БЕЗ ПАРОЛЮ!\n"
"Вы можаце скарыстаць парамэтар загрузкі vncpassword=<пароль>\n"
"калі жадаеце мець бясьпечны паслужнік.\n"
"\n"

#: pyanaconda/vnc.py:261
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:263
#, fuzzy
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr "Невядомая памылка"

#: pyanaconda/vnc.py:285 pyanaconda/vnc.py:378
msgid "VNC Configuration"
msgstr "Наладка VNC"

#: pyanaconda/vnc.py:288 pyanaconda/vnc.py:382
msgid "No password"
msgstr "Без паролю"

#: pyanaconda/vnc.py:290 pyanaconda/vnc.py:385
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""

#: pyanaconda/vnc.py:298 pyanaconda/vnc.py:393
#: pyanaconda/textw/userauth_text.py:47
msgid "Password:"
msgstr "Пароль:"

#: pyanaconda/vnc.py:299 pyanaconda/vnc.py:394
#: pyanaconda/textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr "Пароль (пацьверджаньне):"

#: pyanaconda/vnc.py:317 pyanaconda/vnc.py:416
#: pyanaconda/textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr "Паролі не супадаюць"

#: pyanaconda/vnc.py:318 pyanaconda/vnc.py:417
#: pyanaconda/textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr "Паролі якія вы ўвялі не супадаюц. Калі ласка, паўтарыце спробу."

#: pyanaconda/vnc.py:323 pyanaconda/vnc.py:422
#: pyanaconda/textw/userauth_text.py:66
msgid "Password Length"
msgstr "Даўжыня пароля"

#: pyanaconda/vnc.py:324 pyanaconda/vnc.py:423
msgid "The password must be at least six characters long."
msgstr "Пароль мусіць быць даўжэйшым за пяць знакаў."

#: pyanaconda/vnc.py:346
msgid "Unable to Start X"
msgstr "Немагчыма запусьціць X"

#: pyanaconda/vnc.py:348
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""

#: pyanaconda/vnc.py:367
msgid "Start VNC"
msgstr "Запуск VNC"

#: pyanaconda/vnc.py:368 pyanaconda/vnc.py:370
msgid "Use text mode"
msgstr "Выкарыстоўваць тэкставы рэжым"

#: pyanaconda/yuminstall.py:85
#, python-format
msgid "%s MB"
msgstr "%s Мб"

#: pyanaconda/yuminstall.py:88
#, python-format
msgid "%s KB"
msgstr "%s Кб"

#: pyanaconda/yuminstall.py:90
#, fuzzy, python-format
msgid "%s Byte"
msgid_plural "%s Bytes"
msgstr[0] "%s байт"
msgstr[1] "%s байт"

#: pyanaconda/yuminstall.py:134
msgid "Preparing to install"
msgstr ""

#: pyanaconda/yuminstall.py:135
#, fuzzy
msgid "Preparing transaction from installation source"
msgstr "Падрыхтоўка для ўсталёўваньня..."

#: pyanaconda/yuminstall.py:163
#, fuzzy, python-format
msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
msgstr "_Усталяваць %s"

#: pyanaconda/yuminstall.py:212
#, python-format
msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/yuminstall.py:227
msgid "Finishing upgrade"
msgstr ""

#: pyanaconda/yuminstall.py:228
msgid "Finishing upgrade process.  This may take a little while."
msgstr ""

#: pyanaconda/yuminstall.py:251
#, fuzzy
msgid "Error Installing Package"
msgstr "Усталёўка пакетаў"

#: pyanaconda/yuminstall.py:252
#, python-format
msgid ""
"A fatal error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""

#: pyanaconda/yuminstall.py:402 pyanaconda/iw/task_gui.py:324
#, fuzzy
msgid "Error Setting Up Repository"
msgstr "Нерэчаісная метка загрузкі"

#: pyanaconda/yuminstall.py:403
#, fuzzy, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Please provide the correct information for installing %(productName)s."
msgstr ""
"Пад час аўтаматычнага разбіеньня адбыліся наступныя памылкі:\n"
"\n"
"%s\n"
"\n"
"Націсьніце \"Добра\" для перазагрузкі сыстэмы."

#: pyanaconda/yuminstall.py:448
#, fuzzy
msgid "Change Disc"
msgstr "Зьмяніць"

#: pyanaconda/yuminstall.py:449
#, fuzzy, python-format
msgid "Please insert %(productName)s disc %(discnum)d to continue."
msgstr "Калі ласка, устаўце %s дыск %d для працягу"

#: pyanaconda/yuminstall.py:459
#, fuzzy
msgid "Wrong Disc"
msgstr "Памылковы дыск"

#: pyanaconda/yuminstall.py:460
#, fuzzy, python-format
msgid "That's not the correct %s disc."
msgstr "Гэты дыск ня той; патрэбны %s."

#: pyanaconda/yuminstall.py:467
#, fuzzy
msgid "Unable to access the disc."
msgstr "Немагчыма атрымаць доступ да дыска."

#: pyanaconda/yuminstall.py:631
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""

#: pyanaconda/yuminstall.py:752 pyanaconda/yuminstall.py:1200
#: pyanaconda/iw/task_gui.py:312
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""

#: pyanaconda/yuminstall.py:833 pyanaconda/yuminstall.py:835
#, fuzzy
msgid "Re_boot"
msgstr "Перазагруз_іць"

#: pyanaconda/yuminstall.py:833
#, fuzzy
msgid "_Eject"
msgstr "Вываліць дыск"

#: pyanaconda/yuminstall.py:839
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: pyanaconda/yuminstall.py:887
#, fuzzy
msgid "Retrying"
msgstr "Выцягваньне"

#: pyanaconda/yuminstall.py:887
#, fuzzy
msgid "Retrying download."
msgstr "Пошук пакетаў для абнаўленьня..."

#: pyanaconda/yuminstall.py:952
#, fuzzy, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr ""
"Назва вузла \"%s\" зьяўляецца нерэчаіснай з-за наступнае прычыны\n"
"\n"
"%s"

#: pyanaconda/yuminstall.py:995 pyanaconda/yuminstall.py:996
msgid "file conflicts"
msgstr ""

#: pyanaconda/yuminstall.py:997
#, fuzzy
msgid "older package(s)"
msgstr "Неабавязковыя пакеты"

#: pyanaconda/yuminstall.py:998
msgid "insufficient disk space"
msgstr ""

#: pyanaconda/yuminstall.py:999
msgid "insufficient disk inodes"
msgstr ""

#: pyanaconda/yuminstall.py:1000
msgid "package conflicts"
msgstr ""

#: pyanaconda/yuminstall.py:1001
msgid "package already installed"
msgstr ""

#: pyanaconda/yuminstall.py:1002
msgid "required package"
msgstr ""

#: pyanaconda/yuminstall.py:1003
msgid "package for incorrect arch"
msgstr ""

#: pyanaconda/yuminstall.py:1004
msgid "package for incorrect os"
msgstr ""

#: pyanaconda/yuminstall.py:1018
msgid "You need more space on the following file systems:\n"
msgstr ""

#: pyanaconda/yuminstall.py:1031
#, fuzzy, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr "Носьбіт якога тыпу ўтрымлівае пакеты для ўсталёўваньня?"

#: pyanaconda/yuminstall.py:1034
#, fuzzy, python-format
msgid ""
"There was an error running your transaction for the following reason(s): "
"%s.\n"
msgstr ""
"Назва вузла \"%s\" зьяўляецца нерэчаіснай з-за наступнае прычыны\n"
"\n"
"%s"

#: pyanaconda/yuminstall.py:1041 pyanaconda/yuminstall.py:1046
#, fuzzy
msgid "Error Running Transaction"
msgstr "Падрыхтоўка транзакцыі RPM..."

#: pyanaconda/yuminstall.py:1226
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""

#: pyanaconda/yuminstall.py:1275 pyanaconda/textw/constants_text.py:64
msgid "Edit"
msgstr "Рэдагаваць"

#: pyanaconda/yuminstall.py:1286 pyanaconda/yuminstall.py:1477
#, fuzzy, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""
"Немагчыма прачытаць сьпіс загалоўкаў. Гэта можа здарыцца з-за адсутнічаньня "
"файла альбо пашкоджанага носьбіта. Націсьніце <ўвод> каб паўтарыць спробу."

#: pyanaconda/yuminstall.py:1455
msgid ""
"Some of the packages you have selected for install are missing "
"dependencies.  You can exit the installation, go back and change your "
"package selections, or continue installing these packages without their "
"dependencies.  If you continue, these packages may not work correctly due to "
"missing components."
msgstr ""

#: pyanaconda/yuminstall.py:1499
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""

#: pyanaconda/yuminstall.py:1520
msgid "Reboot?"
msgstr ""

#: pyanaconda/yuminstall.py:1521
msgid "The system will be rebooted now."
msgstr ""

#: pyanaconda/yuminstall.py:1663
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""

#: pyanaconda/yuminstall.py:1698
#, python-format
msgid ""
"The arch of the release of %(productName)s you are upgrading to appears to "
"be %(myarch)s which does not match your previously installed arch of "
"%(arch)s.  This is likely to not succeed.  Are you sure you wish to continue "
"the upgrade process?"
msgstr ""

#: pyanaconda/yuminstall.py:1745
#, fuzzy
msgid "Post Upgrade"
msgstr "Абнаўленьне сыстэмы"

#: pyanaconda/yuminstall.py:1746
#, fuzzy
msgid "Performing post-upgrade configuration"
msgstr "Наладка загрузчыка"

#: pyanaconda/yuminstall.py:1748
#, fuzzy
msgid "Post Installation"
msgstr "Усталёўваньне пакета"

#: pyanaconda/yuminstall.py:1749
#, fuzzy
msgid "Performing post-installation configuration"
msgstr "Наладка загрузчыка"

#: pyanaconda/yuminstall.py:1990
#, fuzzy
msgid "Installation Starting"
msgstr "Пачатак усталёўкі"

#: pyanaconda/yuminstall.py:1991
#, fuzzy
msgid "Starting installation process"
msgstr "Тып усталёўкі"

#: pyanaconda/yuminstall.py:2029
msgid "Dependency Check"
msgstr "Праверка залежнасьцяў"

#: pyanaconda/yuminstall.py:2030
#, fuzzy
msgid "Checking dependencies in packages selected for installation"
msgstr "Праверка залежнасьцяў у пакетах, што вылучаны для ўсталёўваньня..."

#: pyanaconda/yuminstall.py:2086 pyanaconda/yuminstall.py:2102
#, fuzzy
msgid "Retrieving installation information."
msgstr "Чытаньне зьвестак пакетаў..."

#: pyanaconda/yuminstall.py:2088 pyanaconda/yuminstall.py:2104
#, fuzzy, python-format
msgid "Retrieving installation information for %s."
msgstr "Чытаньне зьвестак пакетаў..."

#: pyanaconda/yuminstall.py:2089 pyanaconda/yuminstall.py:2105
#, fuzzy
msgid "Installation Progress"
msgstr "Тып усталёўкі"

#: pyanaconda/installclasses/fedora.py:38
#, fuzzy
msgid "_Fedora"
msgstr "Фармат"

#: pyanaconda/installclasses/fedora.py:39
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. You can optionally select a different set of "
"software now."
msgstr ""

#: pyanaconda/installclasses/fedora.py:48
msgid "Graphical Desktop"
msgstr ""

#: pyanaconda/installclasses/fedora.py:53
msgid "Software Development"
msgstr ""

#: pyanaconda/installclasses/fedora.py:59
#, fuzzy
msgid "Web Server"
msgstr "DNS паслужнік"

#: pyanaconda/installclasses/fedora.py:63 pyanaconda/installclasses/rhel.py:44
msgid "Minimal"
msgstr ""

#: pyanaconda/installclasses/rhel.py:33
msgid "Red Hat Enterprise Linux"
msgstr ""

#: pyanaconda/installclasses/rhel.py:34
#, python-format
msgid ""
"The default installation of %s is a minimum install. You can optionally "
"select a different set of software now."
msgstr ""

#: pyanaconda/iw/GroupSelector.py:147
#, fuzzy, python-format
msgid "Packages in %s"
msgstr "Пакеты"

#: pyanaconda/iw/GroupSelector.py:433
#, fuzzy, python-format
msgid "Optional packages selected: %(inst)d of %(cnt)d"
msgstr "Носьбіт якога тыпу ўтрымлівае пакеты для ўсталёўваньня?"

#: pyanaconda/iw/GroupSelector.py:435
#, python-format
msgid "<i>%s</i>"
msgstr ""

#: pyanaconda/iw/GroupSelector.py:495
msgid "Uncategorized"
msgstr ""

#: pyanaconda/iw/account_gui.py:52
#, fuzzy
msgid "Root _Password:"
msgstr "_Пароль root:"

#: pyanaconda/iw/account_gui.py:54
#, fuzzy
msgid "_Confirm:"
msgstr "Па_цьверджаньне:"

#: pyanaconda/iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr ""

#: pyanaconda/iw/account_gui.py:102 pyanaconda/iw/account_gui.py:110
#: pyanaconda/iw/account_gui.py:117 pyanaconda/iw/account_gui.py:138
#: pyanaconda/textw/userauth_text.py:74
msgid "Error with Password"
msgstr "Памылка з паролем"

#: pyanaconda/iw/account_gui.py:103
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr "Вы мусіце ўвесьці пароль для root і пацьвердзіць яго для працягу."

#: pyanaconda/iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr "Паролі, якія вы ўвялі не супадаюць. Калі ласка, паўтарыце спробу."

#: pyanaconda/iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr "Пароль для root мусіць быць даўжэйшым за 5 знакаў."

#: pyanaconda/iw/account_gui.py:127 pyanaconda/textw/userauth_text.py:83
#, fuzzy
msgid "Weak Password"
msgstr "Пароль"

#: pyanaconda/iw/account_gui.py:128
#, python-format
msgid "You have provided a weak password: %s"
msgstr ""

#: pyanaconda/iw/account_gui.py:131
#, fuzzy
msgid "_Use Anyway"
msgstr "Уліковае імя"

#: pyanaconda/iw/account_gui.py:139 pyanaconda/textw/userauth_text.py:75
#, fuzzy
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""
"Запатрабаваны пароль утрымлівае не ASCII знакі, што забаронена ў дачыненьні "
"для пароляў."

#: pyanaconda/iw/advanced_storage.py:87
#, fuzzy
msgid "You must select a NIC to use."
msgstr "Вы мусіце вылучыць падзел для рэдагаваньня"

#: pyanaconda/iw/advanced_storage.py:132
#, fuzzy
msgid "Invalid Initiator Name"
msgstr "Нерэчаісныя тэчкі"

#: pyanaconda/iw/advanced_storage.py:133
#, fuzzy
msgid "You must provide an initiator name."
msgstr "Вы мусіце задаць назву вузла."

#: pyanaconda/iw/advanced_storage.py:162
#, fuzzy
msgid "Error with Data"
msgstr "Памылка даньняў"

#: pyanaconda/iw/autopart_type.py:96
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""

#: pyanaconda/iw/autopart_type.py:119
#, fuzzy
msgid "Resize FileSystem Error"
msgstr "Фарматаваньне файлавае сыстэмы %s..."

#: pyanaconda/iw/autopart_type.py:120
#, python-format
msgid "%(device)s: %(msg)s"
msgstr ""

#: pyanaconda/iw/autopart_type.py:129
msgid "Resize Device Error"
msgstr ""

#: pyanaconda/iw/autopart_type.py:130
#, python-format
msgid "%(name)s: %(msg)s"
msgstr ""

#: pyanaconda/iw/autopart_type.py:250
#, fuzzy
msgid "Use All Space"
msgstr "Занятая прастора:"

#: pyanaconda/iw/autopart_type.py:252
msgid ""
"Removes all partitions on the selected device(s).  This includes partitions "
"created by other operating systems.\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: pyanaconda/iw/autopart_type.py:258
msgid "Replace Existing Linux System(s)"
msgstr ""

#: pyanaconda/iw/autopart_type.py:260
msgid ""
"Removes all Linux partitions on the selected device(s). This does not remove "
"other partitions you may have on your storage device(s) (such as VFAT or "
"FAT32).\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: pyanaconda/iw/autopart_type.py:268
#, fuzzy
msgid "Shrink Current System"
msgstr "Ключ шыфраваньня"

#: pyanaconda/iw/autopart_type.py:270
msgid ""
"Shrinks existing partitions to create free space for the default layout."
msgstr ""

#: pyanaconda/iw/autopart_type.py:272
#, fuzzy
msgid "Use Free Space"
msgstr "Бракуе вольнае прасторы"

#: pyanaconda/iw/autopart_type.py:274
msgid ""
"Retains your current data and partitions and uses only the unpartitioned "
"space on the selected device(s), assuming you have enough free space "
"available."
msgstr ""

#: pyanaconda/iw/autopart_type.py:278
msgid "Create Custom Layout"
msgstr ""

#: pyanaconda/iw/autopart_type.py:280
msgid ""
"Manually create your own custom layout on the selected device(s) using our "
"partitioning tool."
msgstr ""

#: pyanaconda/iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr "_Выкарыстоўваць пароль на загрузку"

#: pyanaconda/iw/blpasswidget.py:45
#, fuzzy
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""
"Пароль на загрузку стварае абарону ад зьмены парамэнтраў ядру. Каб павысіць "
"узровень бясьпекі вельмі раціца выставіць гэты пароль."

#: pyanaconda/iw/blpasswidget.py:76
msgid "Change _password"
msgstr "Зьмяніць _пароль"

#: pyanaconda/iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr "Увядзіце пароль на загрузку"

#: pyanaconda/iw/blpasswidget.py:105
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:112
msgid "_Password:"
msgstr "_Пароль:"

#: pyanaconda/iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr "Па_цьверджаньне:"

#: pyanaconda/iw/blpasswidget.py:139
msgid "Passwords don't match"
msgstr "Паролі не супадаюць"

#: pyanaconda/iw/blpasswidget.py:140
msgid "Passwords do not match"
msgstr "Паролі не супадаюць"

#: pyanaconda/iw/blpasswidget.py:149
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: pyanaconda/iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr "Наладка загрузчыка"

#: pyanaconda/iw/bootloader_main_gui.py:180
#: pyanaconda/iw/bootloader_main_gui.py:185
#: pyanaconda/iw/bootloader_main_gui.py:223
#, fuzzy, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "Усталёўваньне загрузчыка..."

#: pyanaconda/iw/bootloader_main_gui.py:229
#, fuzzy
msgid "_Change device"
msgstr "Зьмяніць"

#: pyanaconda/iw/cleardisks_gui.py:34
msgid "Clear Disks Selector"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:45 pyanaconda/iw/filter_gui.py:432
#, fuzzy
msgid "You must select at least one drive to be used for installation."
msgstr "Вылучыце адпаведную мышу для сыстэмы."

#: pyanaconda/iw/cleardisks_gui.py:55
#, fuzzy
msgid "You must select one drive to boot from."
msgstr "Вы мусіце вылучыць падзел для рэдагаваньня"

#: pyanaconda/iw/cleardisks_gui.py:127 pyanaconda/iw/cleardisks_gui.py:147
#: pyanaconda/iw/filter_gui.py:489 pyanaconda/iw/filter_gui.py:500
#: pyanaconda/iw/filter_gui.py:530
msgid "Model"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:128 pyanaconda/iw/cleardisks_gui.py:148
#: pyanaconda/iw/filter_gui.py:490 pyanaconda/iw/filter_gui.py:501
#: pyanaconda/iw/filter_gui.py:509 pyanaconda/iw/filter_gui.py:520
#: pyanaconda/iw/filter_gui.py:531
msgid "Capacity"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:129 pyanaconda/iw/filter_gui.py:491
#: pyanaconda/iw/filter_gui.py:510 pyanaconda/iw/filter_gui.py:521
#: pyanaconda/iw/filter_gui.py:532
msgid "Vendor"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:130 pyanaconda/iw/cleardisks_gui.py:149
#: pyanaconda/iw/filter_gui.py:508 pyanaconda/iw/filter_gui.py:519
#: pyanaconda/iw/filter_gui.py:535
msgid "Identifier"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:131 pyanaconda/iw/filter_gui.py:492
#: pyanaconda/iw/filter_gui.py:511 pyanaconda/iw/filter_gui.py:522
#: pyanaconda/iw/filter_gui.py:533
msgid "Interconnect"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:145
#, fuzzy
msgid ""
"Boot\n"
"Loader"
msgstr "Загрузчык"

#: pyanaconda/iw/cleardisks_gui.py:196
msgid ""
"<b>Tip:</b> All Linux filesystems on the install target devices will be "
"reformatted and wiped of any data.  Make sure you have backups."
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:198
msgid ""
"<b>Tip:</b> The install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:200
msgid ""
"<b>Tip:</b> Your filesystems on the install target devices will not be "
"reformatted unless you choose to do so during customization."
msgstr ""

#: pyanaconda/iw/congrats_gui.py:33
msgid "Congratulations"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:74 pyanaconda/textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:77 pyanaconda/textw/complete_text.py:42
msgid "Shutdown"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:79 pyanaconda/textw/complete_text.py:44
#, fuzzy
msgid "Please shutdown to use the installed system.\n"
msgstr "Калі ласка, усталюйце далучэньне да %s каб пачаць усталёўку..."

#: pyanaconda/iw/congrats_gui.py:81 pyanaconda/textw/complete_text.py:46
#, fuzzy
msgid "Please reboot to use the installed system.\n"
msgstr "Калі ласка, усталюйце далучэньне да %s каб пачаць усталёўку..."

#: pyanaconda/iw/congrats_gui.py:86 pyanaconda/textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""

#: pyanaconda/iw/congrats_gui.py:90 pyanaconda/textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""

#: pyanaconda/iw/examine_gui.py:39
msgid "Upgrade Examine"
msgstr ""

#: pyanaconda/iw/examine_gui.py:58
#, fuzzy
msgid "Fresh Installation"
msgstr "Усталёўваньне пакета"

#: pyanaconda/iw/examine_gui.py:60
#, python-format
msgid ""
"Choose this option to install a fresh copy of %s on your system.  Existing "
"software and data may be overwritten depending on your configuration choices."
msgstr ""

#: pyanaconda/iw/examine_gui.py:65
#, fuzzy
msgid "Upgrade an Existing Installation"
msgstr "Усталёўваньне пакета"

#: pyanaconda/iw/examine_gui.py:67
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option will preserve the existing data on your storage device(s)."
msgstr ""

#: pyanaconda/iw/examine_gui.py:117
msgid ""
"At least one existing installation has been detected on your system.  What "
"would you like to do?"
msgstr ""

#: pyanaconda/iw/examine_gui.py:136
#, python-format
msgid "<b>Which %s installation would you like to upgrade?</b>"
msgstr ""

#: pyanaconda/iw/examine_gui.py:150
msgid "Unknown Linux system"
msgstr ""

#: pyanaconda/iw/filter_gui.py:150
#, python-format
msgid "Selected devices: %s (%s MB) out of %s (%s MB)."
msgstr ""

#: pyanaconda/iw/filter_gui.py:400
#, fuzzy
msgid "Device Filter"
msgstr "Выцягваньне"

#: pyanaconda/iw/filter_gui.py:493 pyanaconda/iw/filter_gui.py:523
#: pyanaconda/iw/filter_gui.py:534
msgid "Serial Number"
msgstr ""

#: pyanaconda/iw/filter_gui.py:494 pyanaconda/iw/filter_gui.py:502
#: pyanaconda/iw/filter_gui.py:513 pyanaconda/iw/filter_gui.py:524
#: pyanaconda/iw/filter_gui.py:539 pyanaconda/iw/osbootwidget.py:66
#: pyanaconda/iw/partition_gui.py:611
msgid "Device"
msgstr "Прылада"

#: pyanaconda/iw/filter_gui.py:512
msgid "Paths"
msgstr ""

#: pyanaconda/iw/filter_gui.py:536
msgid "Port"
msgstr ""

#: pyanaconda/iw/filter_gui.py:537
msgid "Target"
msgstr ""

#: pyanaconda/iw/filter_gui.py:538
msgid "LUN"
msgstr ""

#: pyanaconda/iw/filter_type.py:55
#, fuzzy
msgid "What type of devices will your installation involve?"
msgstr "Носьбіт якога тыпу ўтрымлівае выратавальны вобраз?"

#: pyanaconda/iw/filter_type.py:61
#, fuzzy
msgid "Basic Storage Devices"
msgstr "Адсутнічае пакет"

#: pyanaconda/iw/filter_type.py:62
msgid ""
"Installs or upgrades to typical types of storage devices.  If you're not "
"sure which option is right for you, this is probably it."
msgstr ""

#: pyanaconda/iw/filter_type.py:66
#, fuzzy
msgid "Specialized Storage Devices"
msgstr "Адсутнічае пакет"

#: pyanaconda/iw/filter_type.py:67
msgid ""
"Installs or upgrades to enterprise devices such as Storage Area Networks "
"(SANs). This option will allow you to add FCoE / iSCSI / zFCP disks and to "
"filter out devices the installer should ignore."
msgstr ""

#: pyanaconda/iw/language_gui.py:33 pyanaconda/textw/language_text.py:45
msgid "Language Selection"
msgstr "Вылучэньне мовы"

#: pyanaconda/iw/language_gui.py:81 loader/lang.c:371
#: pyanaconda/textw/language_text.py:46
msgid "What language would you like to use during the installation process?"
msgstr "На якой мовы вы жадаеце весьці ўсталёўку?"

#: pyanaconda/iw/lvm_dialog_gui.py:126 pyanaconda/iw/lvm_dialog_gui.py:172
#: pyanaconda/iw/lvm_dialog_gui.py:186 pyanaconda/iw/lvm_dialog_gui.py:225
#: pyanaconda/iw/lvm_dialog_gui.py:302 pyanaconda/iw/lvm_dialog_gui.py:697
#: pyanaconda/iw/lvm_dialog_gui.py:718
msgid "Not enough space"
msgstr "Бракуе прасторы"

#: pyanaconda/iw/lvm_dialog_gui.py:127
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:136
msgid "Confirm Physical Extent Change"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:137
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:146 pyanaconda/iw/lvm_dialog_gui.py:208
msgid "C_ontinue"
msgstr "Пра_цяг"

#: pyanaconda/iw/lvm_dialog_gui.py:173
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is larger than the smallest physical volume "
"(%(maxpvsize)10.2f MB) in the volume group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:187
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is too large compared to the size of the smallest "
"physical volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:201
msgid "Too small"
msgstr "Надта маленькі"

#: pyanaconda/iw/lvm_dialog_gui.py:202
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:226
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:303
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:402
msgid "Make Logical Volume"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:404
#, python-format
msgid "Edit Logical Volume: %s"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:443 pyanaconda/iw/raid_dialog_gui.py:451
msgid "_File System Type:"
msgstr "Тып _файлавае сыстэмы:"

#: pyanaconda/iw/lvm_dialog_gui.py:449
msgid "_Logical Volume Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:458
#: pyanaconda/iw/partition_dialog_gui.py:492
msgid "_Size (MB):"
msgstr "_Памер (Мб):"

#: pyanaconda/iw/lvm_dialog_gui.py:465
#, python-format
msgid "(Max size is %s MB)"
msgstr "(Найбольшы памер %s Мб)"

#: pyanaconda/iw/lvm_dialog_gui.py:470
#: pyanaconda/iw/partition_dialog_gui.py:544
#: pyanaconda/iw/partition_ui_helpers_gui.py:337
#: pyanaconda/iw/raid_dialog_gui.py:423
#, fuzzy
msgid "_Encrypt"
msgstr "Ключ шыфраваньня"

#: pyanaconda/iw/lvm_dialog_gui.py:479
#: pyanaconda/iw/partition_dialog_gui.py:474
#: pyanaconda/iw/raid_dialog_gui.py:461
msgid "Original File System Type:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:484
#: pyanaconda/iw/partition_dialog_gui.py:482
#: pyanaconda/iw/raid_dialog_gui.py:468
msgid "Original File System Label:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:489
msgid "Logical Volume Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:493
msgid "Size (MB):"
msgstr "Памер (Мб):"

#: pyanaconda/iw/lvm_dialog_gui.py:520
#: pyanaconda/iw/partition_dialog_gui.py:430
#: pyanaconda/iw/raid_dialog_gui.py:442
msgid "_Mount Point:"
msgstr "Пункт _мацаваньня:"

#: pyanaconda/iw/lvm_dialog_gui.py:601
#: pyanaconda/iw/partition_dialog_gui.py:118
#: pyanaconda/iw/raid_dialog_gui.py:176
#, fuzzy
msgid "Mount Point Error"
msgstr "Пункт _мацаваньня:"

#: pyanaconda/iw/lvm_dialog_gui.py:611
msgid "Illegal Logical Volume Name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:626
msgid "Illegal logical volume name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:627
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:668
#: pyanaconda/iw/partition_dialog_gui.py:133
#: pyanaconda/iw/raid_dialog_gui.py:191
msgid "Mount point in use"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:669
#: pyanaconda/iw/partition_dialog_gui.py:134
#: pyanaconda/iw/raid_dialog_gui.py:192
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:684
msgid "Illegal size"
msgstr "Недапушчальны памер"

#: pyanaconda/iw/lvm_dialog_gui.py:685
msgid "The requested size as entered is not a valid number greater than 0."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:698
#, python-format
msgid ""
"The current requested size (%(size)10.2f MB) is larger than the maximum "
"logical volume size (%(maxlv)10.2f MB). To increase this limit you can "
"create more Physical Volumes from unpartitioned disk space and add them to "
"this Volume Group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:719
#, python-format
msgid ""
"The size entered for this logical volume (%(size)d MB) combined with the "
"size of the other logical volume(s) exceeds the size of the volume group "
"(%(tempvgsize)d MB). Please make the volume group larger or make the logical "
"volume smaller."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:829
msgid "No free slots"
msgstr "Няма вольных слотаў"

#: pyanaconda/iw/lvm_dialog_gui.py:830
#, python-format
msgid "You cannot create more than %d logical volume per volume group."
msgid_plural "You cannot create more than %d logical volumes per volume group."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/iw/lvm_dialog_gui.py:840
msgid "No free space"
msgstr "Бракуе вольнае прасторы"

#: pyanaconda/iw/lvm_dialog_gui.py:841
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:878
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:984
msgid "Invalid Volume Group Name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:993
msgid "Name in use"
msgstr "Назва ўжо выкарыстоўваецца"

#: pyanaconda/iw/lvm_dialog_gui.py:994
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1281
msgid "Not enough physical volumes"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1282
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1293
msgid "Make LVM Volume Group"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1296
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1298
msgid "Edit LVM Volume Group"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1314
msgid "_Volume Group Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1322
msgid "Volume Group Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1330
msgid "_Physical Extent:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1345
msgid "Physical Volumes to _Use:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1351
msgid "Used Space:"
msgstr "Занятая прастора:"

#: pyanaconda/iw/lvm_dialog_gui.py:1368
msgid "Free Space:"
msgstr "Вольная прастора:"

#: pyanaconda/iw/lvm_dialog_gui.py:1386
msgid "Total Space:"
msgstr "Агульная прастора:"

#: pyanaconda/iw/lvm_dialog_gui.py:1424
msgid "Logical Volume Name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1427 pyanaconda/iw/partition_gui.py:614
#: pyanaconda/iw/upgrade_swap_gui.py:138 pyanaconda/textw/upgrade_text.py:124
msgid "Mount Point"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1430 pyanaconda/iw/partition_gui.py:613
msgid "Size (MB)"
msgstr "Памер (Мб)"

#: pyanaconda/iw/lvm_dialog_gui.py:1444 pyanaconda/iw/osbootwidget.py:96
msgid "_Add"
msgstr "Д_адаць"

#: pyanaconda/iw/lvm_dialog_gui.py:1447 pyanaconda/iw/osbootwidget.py:100
#: pyanaconda/iw/partition_gui.py:1811 pyanaconda/iw/partition_gui.py:1821
msgid "_Edit"
msgstr "Р_эдагаваць"

#: pyanaconda/iw/lvm_dialog_gui.py:1462
msgid "_Logical Volumes"
msgstr ""

#: pyanaconda/iw/netconfig_dialog.py:192 pyanaconda/textw/netconfig_text.py:36
#, fuzzy, python-format
msgid ""
"An error occurred converting the value entered for \"%(field)s\":\n"
"%(errmsg)s"
msgstr ""
"Узьнікла памылка пераўтварэньня ўведзенага значэньня для \"%s\":\n"
"%s"

#: pyanaconda/iw/netconfig_dialog.py:195 pyanaconda/iw/netconfig_dialog.py:204
#: pyanaconda/textw/netconfig_text.py:35 pyanaconda/textw/netconfig_text.py:42
msgid "Error With Data"
msgstr "Памылка даньняў"

#: pyanaconda/iw/netconfig_dialog.py:203
#, fuzzy, python-format
msgid "A value is required for the field %s."
msgstr "Значэньне неабходня для поля \"%s\"."

#: pyanaconda/iw/netconfig_dialog.py:215
#, fuzzy
msgid "Error Configuring Network"
msgstr "Наладка сеткі для %s"

#: pyanaconda/iw/netconfig_dialog.py:241
#, fuzzy
msgid "Dynamic IP Address"
msgstr "Дынамічны IP"

#: pyanaconda/iw/netconfig_dialog.py:242
#, python-format
msgid "Sending request for IP address information for %s"
msgstr ""

#: pyanaconda/iw/netconfig_dialog.py:257 pyanaconda/iw/netconfig_dialog.py:260
msgid "IP Address"
msgstr "АйПі Адрас"

#: pyanaconda/iw/netconfig_dialog.py:267
#: pyanaconda/textw/netconfig_text.py:290
#, fuzzy
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "IP адрас мусіць утрымліваць ліжбы ў прамежку ад 1 да 255"

#: pyanaconda/iw/netconfig_dialog.py:268 pyanaconda/iw/netconfig_dialog.py:274
#: pyanaconda/iw/netconfig_dialog.py:282 pyanaconda/iw/netconfig_dialog.py:285
#, fuzzy
msgid "IPv4 Network Mask"
msgstr "Адрас/маска"

#: pyanaconda/iw/netconfig_dialog.py:295
msgid "Gateway"
msgstr "Брама"

#: pyanaconda/iw/netconfig_dialog.py:305
msgid "Nameserver"
msgstr "DNS паслужнік"

#: pyanaconda/iw/netconfig_dialog.py:314
msgid "Error configuring network device:"
msgstr ""

#: pyanaconda/iw/network_gui.py:83 pyanaconda/iw/network_gui.py:89
#, fuzzy
msgid "Error with Hostname"
msgstr "Памылка даньняў"

#: pyanaconda/iw/network_gui.py:84
#, fuzzy
msgid "You must enter a valid hostname for this computer."
msgstr "Вы мусіце ўвесьці рэчаісныя IP зьвесткі для працягу"

#: pyanaconda/iw/network_gui.py:90
#, fuzzy, python-format
msgid ""
"The hostname \"%(hostname)s\" is not valid for the following reason:\n"
"\n"
"%(herrors)s"
msgstr ""
"Назва вузла \"%s\" зьяўляецца нерэчаіснай з-за наступнае прычыны\n"
"\n"
"%s"

#: pyanaconda/iw/network_gui.py:134
#, fuzzy
msgid "Select network interface"
msgstr "Вы мусіце вылучыць падзел для рэдагаваньня"

#: pyanaconda/iw/network_gui.py:141 data/ui/netconfig.glade.h:10
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""

#: pyanaconda/iw/network_gui.py:211
#, fuzzy
msgid "Select APs"
msgstr "Вылучыць _усё"

#: pyanaconda/iw/network_gui.py:218
msgid "Select APs for wireless devices"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:66
msgid "Default"
msgstr "Дапомна"

#: pyanaconda/iw/osbootwidget.py:66 pyanaconda/iw/partition_gui.py:612
msgid "Label"
msgstr "Метка"

#: pyanaconda/iw/osbootwidget.py:130
msgid "Image"
msgstr "Вобраз"

#: pyanaconda/iw/osbootwidget.py:137
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:145
msgid "_Label"
msgstr "_Метка"

#: pyanaconda/iw/osbootwidget.py:153
msgid "_Device"
msgstr "_Прылада"

#: pyanaconda/iw/osbootwidget.py:183
msgid "Default Boot _Target"
msgstr "Дапомная _мэта загрузкі"

#: pyanaconda/iw/osbootwidget.py:212
msgid "You must specify a label for the entry"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:221
msgid "Boot label contains illegal characters"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:245
msgid "Duplicate Label"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:246
msgid "This label is already in use for another boot entry."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:259
msgid "Duplicate Device"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:260
msgid "This device is already being used for another boot entry."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:322
msgid "Cannot Delete"
msgstr "Немагчыма выдаліць"

#: pyanaconda/iw/osbootwidget.py:323
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:58
msgid "Additional Size Options"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:63
msgid "_Fixed size"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:65
msgid "Fill all space _up to (MB):"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:72
msgid "Fill to maximum _allowable size"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:191
#, fuzzy
msgid "Error With Request"
msgstr "Памылка даньняў"

#: pyanaconda/iw/partition_dialog_gui.py:400
msgid "Add Partition"
msgstr "Дадаць падзел"

#: pyanaconda/iw/partition_dialog_gui.py:402
#, fuzzy, python-format
msgid "Edit Partition: %s"
msgstr "Рэдагаваньне падзела: /dev/%s"

#: pyanaconda/iw/partition_dialog_gui.py:439
msgid "File System _Type:"
msgstr "_Тып файлавае сыстэмы"

#: pyanaconda/iw/partition_dialog_gui.py:455
msgid "Allowable _Drives:"
msgstr "Дазволеныя _прылады:"

#: pyanaconda/iw/partition_dialog_gui.py:531
msgid "Force to be a _primary partition"
msgstr ""

#: pyanaconda/iw/partition_gui.py:360
#, python-format
msgid "Drive %(drive)s (%(size)-0.f MB) (Model: %(model)s)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:400 pyanaconda/iw/partition_gui.py:446
#: pyanaconda/iw/partition_gui.py:513 pyanaconda/iw/partition_gui.py:998
#: pyanaconda/iw/partition_gui.py:1070
msgid "Free"
msgstr ""

#: pyanaconda/iw/partition_gui.py:478
#, python-format
msgid "LVM Volume Group %s (%-0.f MB)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:544
#, python-format
msgid "MD RAID ARRAY %s (%-0.f MB)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:615
msgid "Type"
msgstr "Тып"

#: pyanaconda/iw/partition_gui.py:616 pyanaconda/storage/__init__.py:1817
msgid "Format"
msgstr "Фармат"

#: pyanaconda/iw/partition_gui.py:653
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""

#: pyanaconda/iw/partition_gui.py:655
msgid ""
"Size\n"
"(MB)"
msgstr ""
"Памер\n"
"(Мб)"

#: pyanaconda/iw/partition_gui.py:760
msgid "Partitioning"
msgstr ""

#: pyanaconda/iw/partition_gui.py:850
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""

#: pyanaconda/iw/partition_gui.py:852
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""

#: pyanaconda/iw/partition_gui.py:858
msgid "Partitioning Errors"
msgstr ""

#: pyanaconda/iw/partition_gui.py:865
msgid "The partitioning scheme you requested generated the following warnings."
msgstr ""

#: pyanaconda/iw/partition_gui.py:867
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: pyanaconda/iw/partition_gui.py:872
msgid "Partitioning Warnings"
msgstr ""

#: pyanaconda/iw/partition_gui.py:881
msgid ""
"The following pre-existing devices have been selected to be formatted, "
"destroying all data."
msgstr ""

#: pyanaconda/iw/partition_gui.py:893
msgid "Format Warnings"
msgstr ""

#: pyanaconda/iw/partition_gui.py:986
msgid "LVM Volume Groups"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1008
msgid "RAID Devices"
msgstr "RAID прылады"

#: pyanaconda/iw/partition_gui.py:1024 loader/hdinstall.c:212
msgid "Hard Drives"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1074
msgid "Extended"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1364
msgid "Cannot perform any creation action"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1365
msgid ""
"Note that the creation action requires one of the following:\n"
"\n"
"* Free space in one of the Hard Drives.\n"
"* At least two free Software RAID partitions.\n"
"* At least one free physical volume (LVM) partition.\n"
"* At least one Volume Group with free space."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1468
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1474
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1478
#, python-format
msgid "You currently have %d software RAID partition free to use."
msgid_plural "You currently have %d software RAID partitions free to use."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/iw/partition_gui.py:1482
msgid "About RAID"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1488
#, python-format
msgid ""
"Logical Volume Manager (LVM) is a 3 level construct. The first level is made "
"up of disks or partitions formatted with LVM metadata called Physical "
"Volumes (PV).  A Volume Group (VG) sits on top of one or more PVs. The VG, "
"in turn, is the base to create one or more Logical Volumes (LV).  Note that "
"a VG can be an aggregate of PVs from multiple physical disks.  For more "
"information on using LVM please consult the %s documentation\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1496
msgid ""
"To create a PV you need a partition with free space.  To create a VG you "
"need a PV that is not part of any existing VG.  To create an LV you need a "
"VG with free space.\n"
"\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1500
#, python-format
msgid "You currently have %d available PV free to use.\n"
msgid_plural "You currently have %d available PVs free to use.\n"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/iw/partition_gui.py:1504
msgid "You currently have free space to create PVs."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1507
msgid "About LVM"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1585 pyanaconda/storage/partitioning.py:203
#: pyanaconda/storage/partitioning.py:246
msgid "Error Partitioning"
msgstr "Памылка стварэньня падзелаў"

#: pyanaconda/iw/partition_gui.py:1586
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1595
#, python-format
msgid "Warning: %s."
msgstr "Увага! %s."

#: pyanaconda/iw/partition_gui.py:1626
msgid "Unable To Edit"
msgstr "Немагчыма рэдагаваць"

#: pyanaconda/iw/partition_gui.py:1627
#, fuzzy, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"Вы ня можаце рэдагавць гэты падзел:\n"
"\n"

#: pyanaconda/iw/partition_gui.py:1810
msgid "_Create"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1813
msgid "Re_set"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1836
#, fuzzy
msgid "Please Select A Device"
msgstr "Вылучыць усё"

#: pyanaconda/iw/partition_ui_helpers_gui.py:110
#: pyanaconda/iw/partition_ui_helpers_gui.py:132
#: pyanaconda/iw/partition_ui_helpers_gui.py:134
msgid "<Not Applicable>"
msgstr ""

#: pyanaconda/iw/partition_ui_helpers_gui.py:318
#, fuzzy
msgid "_Format as:"
msgstr "Фарматаваць як:"

#: pyanaconda/iw/partition_ui_helpers_gui.py:340
#, fuzzy
msgid "Mi_grate filesystem to:"
msgstr "Міграваць на:"

#: pyanaconda/iw/partition_ui_helpers_gui.py:370
#, fuzzy
msgid "_Resize"
msgstr "Скінуць"

#: pyanaconda/iw/partition_ui_helpers_gui.py:425
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:390
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:404
msgid "Make RAID Device"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:407
#, fuzzy, python-format
msgid "Edit RAID Device: %s"
msgstr "RAID прылада %s"

#: pyanaconda/iw/raid_dialog_gui.py:409
msgid "Edit RAID Device"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:476
msgid "RAID _Device:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:494
msgid "RAID _Level:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:536
msgid "_RAID Members:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:553
msgid "Number of _spares:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:563
msgid "_Format partition?"
msgstr ""

#: pyanaconda/iw/task_gui.py:55
#, fuzzy, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""
"Немагчыма прачытаць сьпіс загалоўкаў. Гэта можа здарыцца з-за адсутнічаньня "
"файла альбо пашкоджанага носьбіта. Націсьніце <ўвод> каб паўтарыць спробу."

#: pyanaconda/iw/task_gui.py:129
#, fuzzy
msgid "Edit Repository"
msgstr "Нерэчаісная метка загрузкі"

#: pyanaconda/iw/task_gui.py:153
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""

#: pyanaconda/iw/task_gui.py:239
#, fuzzy
msgid "Invalid Proxy URL"
msgstr "Нерэчаісная метка загрузкі"

#: pyanaconda/iw/task_gui.py:240
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr ""

#: pyanaconda/iw/task_gui.py:252 pyanaconda/iw/task_gui.py:436
#, fuzzy
msgid "Invalid Repository URL"
msgstr "Нерэчаісная метка загрузкі"

#: pyanaconda/iw/task_gui.py:253 pyanaconda/iw/task_gui.py:437
#, fuzzy
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr "Вы мусіце задаць назву вузла."

#: pyanaconda/iw/task_gui.py:274 pyanaconda/iw/task_gui.py:446
#, fuzzy
msgid "No Media Found"
msgstr "Кіроўца ня знойдзены"

#: pyanaconda/iw/task_gui.py:275 pyanaconda/iw/task_gui.py:447
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""

#: pyanaconda/iw/task_gui.py:306 pyanaconda/iw/task_gui.py:470
#, fuzzy
msgid "Please enter an NFS server and path."
msgstr "Калі ласка, вылучыце манітор, які падключаны да сыстэмы."

#: pyanaconda/iw/task_gui.py:325
#, fuzzy, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""
"Наступныя памылкі былі знойдзены цягам разбору наладкі kickstart:\n"
"\n"
"%s"

#: pyanaconda/iw/task_gui.py:348
#, fuzzy
msgid "Invalid Repository Name"
msgstr "Нерэчаісная назва вузла"

#: pyanaconda/iw/task_gui.py:349
#, fuzzy
msgid "You must provide a repository name."
msgstr "Вы мусіце задаць назву вузла."

#: pyanaconda/iw/task_gui.py:502 data/ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr ""

#: pyanaconda/iw/task_gui.py:507
msgid "No Software Repos Enabled"
msgstr ""

#: pyanaconda/iw/task_gui.py:508
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""

#: pyanaconda/iw/timezone_gui.py:63 pyanaconda/textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:37
#: pyanaconda/textw/upgrade_bootloader_text.py:129
msgid "Upgrade Boot Loader Configuration"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:120
msgid "_Update boot loader configuration"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:121
msgid "This will update your current boot loader."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:125
#: pyanaconda/textw/upgrade_bootloader_text.py:103
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:128
#: pyanaconda/textw/upgrade_bootloader_text.py:107
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:135
#: pyanaconda/textw/upgrade_bootloader_text.py:116
#, python-format
msgid ""
"The installer has detected the %(type)s boot loader currently installed on "
"%(bootDev)s."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:139
msgid "This is the recommended option."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:144
msgid "_Create new boot loader configuration"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:146
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:153
msgid "_Skip boot loader updating"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:154
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:165
msgid "What would you like to do?"
msgstr ""

#: pyanaconda/iw/upgrade_migratefs_gui.py:38
#: pyanaconda/textw/upgrade_text.py:42
msgid "Migrate File Systems"
msgstr ""

#: pyanaconda/iw/upgrade_migratefs_gui.py:69
#: pyanaconda/textw/upgrade_text.py:44
#, python-format
msgid ""
"This release of %(productName)s supports an updated file system, which has "
"several benefits over the file system traditionally shipped in "
"%(productName)s.  This installation program can migrate formatted partitions "
"without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:92 pyanaconda/textw/upgrade_text.py:107
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have "
"%dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:99
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:111
msgid "I _want to create a swap file"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:120
msgid "Select the _partition to put the swap file on:"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:138 pyanaconda/textw/upgrade_text.py:125
msgid "Partition"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:138
msgid "Free Space (MB)"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:156
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:171
msgid "Swap file _size (MB):"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:181
msgid "I _don't want to create a swap file"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:191
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:199 pyanaconda/textw/upgrade_text.py:194
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:206 pyanaconda/textw/upgrade_text.py:189
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr ""

#: pyanaconda/iw/welcome_gui.py:56 pyanaconda/textw/welcome_text.py:36
msgid "Network Install Required"
msgstr ""

#: pyanaconda/iw/welcome_gui.py:57 pyanaconda/textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no network "
"devices were found on your system.  To avoid a network installation, boot "
"with the full DVD, full CD set, or do not pass a repo= parameter that "
"specifies a network source."
msgstr ""

#: pyanaconda/iw/welcome_gui.py:67
#, fuzzy
msgid "E_xit Installer"
msgstr "Рэдагаваць карыстальніка"

#: pyanaconda/iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr ""

#: pyanaconda/iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr ""

#: pyanaconda/iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""

#: pyanaconda/iw/zipl_gui.py:90 pyanaconda/textw/zipl_text.py:72
msgid "Kernel Parameters"
msgstr "Парамэтры ядра"

#: pyanaconda/iw/zipl_gui.py:93 pyanaconda/iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr "Парамэтры Chandev"

#: loader/net.h:27 pyanaconda/textw/netconfig_text.py:164
#, fuzzy
msgid "Dynamic IP configuration (DHCP)"
msgstr "Наладка SILO"

#: loader/net.h:28
#, fuzzy
msgid "Manual configuration"
msgstr "Наладка z/IPL"

#: loader/net.h:30
#, fuzzy
msgid "Dynamic IP configuration (DHCPv6)"
msgstr "Наладка SILO"

#: loader/net.h:31 pyanaconda/textw/netconfig_text.py:201
msgid "Automatic neighbor discovery"
msgstr ""

#: loader/cdinstall.c:186 loader/cdinstall.c:207 loader/mediacheck.c:61
msgid "Media Check"
msgstr ""

#: loader/cdinstall.c:186 loader/cdinstall.c:189 loader/cdinstall.c:207
#: loader/cdinstall.c:215 loader/method.c:333
msgid "Test"
msgstr "Тэст"

#: loader/cdinstall.c:186 loader/cdinstall.c:190
#, fuzzy
msgid "Eject Disc"
msgstr "Вываліць дыск"

#: loader/cdinstall.c:187
#, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""

#: loader/cdinstall.c:208
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""

#: loader/cdinstall.c:230
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""

#: loader/cdinstall.c:249
#, fuzzy
msgid "Disc Found"
msgstr "Дыск знойдзен"

#: loader/cdinstall.c:250
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""

#: loader/cdinstall.c:331
#, fuzzy
msgid "Scanning"
msgstr "Увага"

#: loader/cdinstall.c:331
#, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr ""

#: loader/cdinstall.c:333
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr ""

#: loader/cdinstall.c:425
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""

#: loader/cdinstall.c:431
#, fuzzy
msgid "Disc Not Found"
msgstr "Дыск ня знойдзен"

#: loader/cdinstall.c:432 loader/driverdisk.c:409 loader/driverdisk.c:441
#: loader/driverdisk.c:522 loader/driverselect.c:79 loader/driverselect.c:153
#: loader/driverselect.c:179 loader/hdinstall.c:212 loader/hdinstall.c:266
#: loader/kbd.c:119 loader/loader.c:510 loader/loader.c:527
#: loader/loader.c:1399 loader/loader.c:1432 loader/net.c:621
#: loader/net.c:1001 loader/net.c:1981 loader/net.c:2000
#: loader/nfsinstall.c:116 loader/urls.c:269
#: pyanaconda/storage/__init__.py:123 pyanaconda/storage/__init__.py:169
#: pyanaconda/storage/devicetree.py:90 pyanaconda/textw/constants_text.py:52
msgid "Back"
msgstr ""

#: loader/cdinstall.c:506
msgid "Cannot find kickstart file on CDROM."
msgstr ""

#: loader/copy.c:51 loader/method.c:287
#, fuzzy, c-format
msgid "Failed to read directory %s: %m"
msgstr "Немагчыма прачытаць тэчку %s: %s"

#: loader/driverdisk.c:274
msgid "Loading"
msgstr "Загрузка"

#: loader/driverdisk.c:274
#, fuzzy
msgid "Reading driver disk"
msgstr "Ужыць дыск с кіроўцам"

#: loader/driverdisk.c:403 loader/driverdisk.c:436
msgid "Driver Disk Source"
msgstr ""

#: loader/driverdisk.c:404
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""

#: loader/driverdisk.c:437
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""

#: loader/driverdisk.c:472
msgid "Failed to mount partition."
msgstr ""

#: loader/driverdisk.c:477
msgid "Select driver disk image"
msgstr ""

#: loader/driverdisk.c:478
msgid "Select the file which is your driver disk image."
msgstr ""

#: loader/driverdisk.c:507
msgid "Failed to load driver disk from file."
msgstr ""

#: loader/driverdisk.c:519
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""

#: loader/driverdisk.c:522
msgid "Insert Driver Disk"
msgstr ""

#: loader/driverdisk.c:535
msgid "Failed to mount driver disk."
msgstr ""

#: loader/driverdisk.c:543
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr ""

#: loader/driverdisk.c:609
msgid "Manually choose"
msgstr ""

#: loader/driverdisk.c:610
msgid "Load another disk"
msgstr ""

#: loader/driverdisk.c:611
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""

#: loader/driverdisk.c:649
msgid "Driver disk"
msgstr ""

#: loader/driverdisk.c:650
msgid "Do you have a driver disk?"
msgstr ""

#: loader/driverdisk.c:659
msgid "More Driver Disks?"
msgstr ""

#: loader/driverdisk.c:660
msgid "Do you wish to load any more driver disks?"
msgstr ""

#: loader/driverdisk.c:709 loader/driverdisk.c:748 loader/hdinstall.c:360
#: loader/kickstart.c:135 loader/kickstart.c:145 loader/kickstart.c:187
#: loader/kickstart.c:192 loader/kickstart.c:339 loader/kickstart.c:355
#: loader/kickstart.c:571 loader/net.c:1654 loader/net.c:1675
#: loader/nfsinstall.c:414 loader/urlinstall.c:374 loader/urlinstall.c:385
#: loader/urlinstall.c:392
msgid "Kickstart Error"
msgstr ""

#: loader/driverdisk.c:710
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr ""

#: loader/driverdisk.c:749
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s"
msgstr ""

#: loader/driverselect.c:68
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""

#: loader/driverselect.c:89
msgid "Enter Module Parameters"
msgstr ""

#: loader/driverselect.c:152
msgid "No drivers found"
msgstr ""

#: loader/driverselect.c:152
msgid "Load driver disk"
msgstr ""

#: loader/driverselect.c:153
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""

#: loader/driverselect.c:171
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""

#: loader/driverselect.c:180
msgid "Specify optional module arguments"
msgstr ""

#: loader/driverselect.c:205
msgid "Select Device Driver to Load"
msgstr ""

#: loader/hdinstall.c:116
#, fuzzy
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""
"Узьнікла памылка чытаньня ўсталёўкі з ISO вобразаў. Калі ласка, праверце "
"вашыя ISO вобразы й паўтарыце спробу."

#: loader/hdinstall.c:213
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""

#: loader/hdinstall.c:226
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""

#: loader/hdinstall.c:250
msgid "Directory holding image:"
msgstr ""

#: loader/hdinstall.c:278
msgid "Select Partition"
msgstr "Вылучыце падзел"

#: loader/hdinstall.c:326
#, fuzzy, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr "Прылада %s ня ўтрымлівае %s вобразы CDROM."

#: loader/hdinstall.c:361
#, c-format
msgid "Bad argument to HD kickstart method command: %s"
msgstr ""

#: loader/hdinstall.c:441 loader/hdinstall.c:497
msgid "Cannot find kickstart file on hard drive."
msgstr ""

#: loader/hdinstall.c:484
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr ""

#: loader/kbd.c:117
msgid "Keyboard Type"
msgstr "Тып клявіятуры"

#: loader/kbd.c:118
msgid "What type of keyboard do you have?"
msgstr ""

#: loader/kickstart.c:136
#, fuzzy, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "Памылка разбору наладкі Kickstart"

#: loader/kickstart.c:146
#, fuzzy, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "Памылка ў %s, радок %d файла kickstart %s."

#: loader/kickstart.c:188
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "Памылка ў %s, радок %d файла kickstart %s."

#: loader/kickstart.c:193
#, fuzzy, c-format
msgid "Missing options on line %d of kickstart file %s."
msgstr "Памылка ў %s, радок %d файла kickstart %s."

#: loader/kickstart.c:300
msgid "Cannot find ks.cfg on removable media."
msgstr ""

#: loader/kickstart.c:340
#, c-format
msgid "Bad argument to device kickstart method command: %s"
msgstr ""

#: loader/kickstart.c:356
msgid "A module name must be specified for the kickstart device command."
msgstr ""

#: loader/kickstart.c:387
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""

#: loader/kickstart.c:396
#, fuzzy
msgid "Error downloading kickstart file"
msgstr "Памылка разбору наладкі Kickstart"

#: loader/kickstart.c:572
#, c-format
msgid "Bad argument to shutdown kickstart method command: %s"
msgstr ""

#: loader/lang.c:63 loader/loader.c:227
#, fuzzy, c-format
msgid "Welcome to %s for %s"
msgstr "Вітаем у %s"

#: loader/lang.c:64
#, fuzzy, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "Вітаем у %s"

#: loader/lang.c:65 loader/loader.c:251
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""
"  <Tab>/<Alt-Tab> паміж элемэнтамі | <Прагал> вылучыць | <F12> наступны "
"экран "

#: loader/lang.c:370
msgid "Choose a Language"
msgstr "Абярыце мову"

#: loader/loader.c:131
#, fuzzy
msgid "Local CD/DVD"
msgstr "Мясцовы CDROM"

#: loader/loader.c:132
msgid "Hard drive"
msgstr "Жорсткі дыск"

#: loader/loader.c:133
#, fuzzy
msgid "NFS directory"
msgstr "%s тэчка:"

#: loader/loader.c:464 loader/loader.c:505
msgid "Update Disk Source"
msgstr "Абнавіць крыніцу"

#: loader/loader.c:465
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""

#: loader/loader.c:506
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""

#: loader/loader.c:524
#, fuzzy, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr ""
"Устаўце дыск з абнаўленьнямі ў /dev/%s і націсьніце \"Добра\" для працягу."

#: loader/loader.c:527
msgid "Updates Disk"
msgstr "Дыск з абнаўленьнямі"

#: loader/loader.c:545
msgid "Failed to mount updates disk"
msgstr ""

#: loader/loader.c:550
msgid "Updates"
msgstr "Абнаўленьні"

#: loader/loader.c:550
#, fuzzy
msgid "Reading anaconda updates"
msgstr "Анаконда працуе праз telnet..."

#: loader/loader.c:586
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""

#: loader/loader.c:595
#, fuzzy
msgid "Error downloading updates image"
msgstr "Памылка разбору наладкі Kickstart"

#: loader/loader.c:1196
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "Бракуе памяці для ўсталёўкі %s на гэай машыне."

#: loader/loader.c:1197
#, fuzzy, c-format
msgid ""
"You do not have enough RAM to install %s on this machine using this install "
"method."
msgstr "Бракуе памяці для ўсталёўкі %s на гэай машыне."

#: loader/loader.c:1262
msgid "Media Detected"
msgstr ""

#: loader/loader.c:1263
#, fuzzy
msgid "Found local installation media"
msgstr "%s %s усталёўка"

#: loader/loader.c:1391
msgid "Rescue Method"
msgstr "Мэтад аднаўленьня"

#: loader/loader.c:1392
msgid "Installation Method"
msgstr "Мэтад усталёўкі"

#: loader/loader.c:1394
msgid "What type of media contains the rescue image?"
msgstr "Носьбіт якога тыпу ўтрымлівае выратавальны вобраз?"

#: loader/loader.c:1396
#, fuzzy
msgid "What type of media contains the installation image?"
msgstr "Носьбіт якога тыпу ўтрымлівае выратавальны вобраз?"

#: loader/loader.c:1431
msgid "No driver found"
msgstr "Кіроўца ня знойдзены"

#: loader/loader.c:1431
msgid "Select driver"
msgstr "Вылучыце кіроўцу"

#: loader/loader.c:1432
msgid "Use a driver disk"
msgstr "Ужыць дыск с кіроўцам"

#: loader/loader.c:1433
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""

#: loader/loader.c:1664
msgid "The following devices have been found on your system."
msgstr "Наступныя прылады былі знойдзены ў вашае сыстэме."

#: loader/loader.c:1666
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""

#: loader/loader.c:1670
msgid "Devices"
msgstr "Прылады"

#: loader/loader.c:1671
msgid "Done"
msgstr "Зроблена"

#: loader/loader.c:1672
msgid "Add Device"
msgstr "Дадаць прыладу"

#: loader/loader.c:1977
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr ""

#: loader/loader.c:2121
#, c-format
msgid "Driver disc was detected in %s. Do you want to use it?."
msgstr ""

#: loader/loader.c:2124
msgid "Driver disc detected"
msgstr ""

#: loader/loader.c:2124
msgid "Use it"
msgstr ""

#: loader/loader.c:2124
#, fuzzy
msgid "Skip it"
msgstr "Абмінуць"

#: loader/loader.c:2459
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr ""

#: loader/loader.c:2461
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr ""

#: loader/mediacheck.c:47
#, c-format
msgid "Unable to find install image %s"
msgstr "Немагчыма адшукаць вобраз усталёўкі %s"

#: loader/mediacheck.c:53
#, fuzzy, c-format
msgid "Checking \"%s\"."
msgstr "Праверка \"%s\"..."

#: loader/mediacheck.c:55
#, fuzzy, c-format
msgid "Checking media."
msgstr "Праверка \"%s\"..."

#: loader/mediacheck.c:87
msgid ""
"Unable to find the checksum in the image.  This probably means the disc was "
"created without adding the checksum."
msgstr ""

#: loader/mediacheck.c:94
#, fuzzy
msgid "Unable to open the image."
msgstr "Немагчыма адшукаць вобраз усталёўкі %s"

#: loader/mediacheck.c:98
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""

#: loader/mediacheck.c:106
msgid "Success"
msgstr ""

#: loader/mediacheck.c:107
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""

#: loader/method.c:330
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"Ці жадаеце падлічыць праверучю суму ISO вобраза:\n"
"\n"
"   %s?"

#: loader/method.c:333
msgid "Checksum Test"
msgstr "Падлік праверчае сумы"

#: loader/net.c:110
#, fuzzy
msgid "Invalid Prefix"
msgstr "Нерэчаісны радок IP адраса"

#: loader/net.c:111
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""

#: loader/net.c:506 loader/net.c:514 loader/net.c:569
#, c-format
msgid "There was an error configuring your network interface."
msgstr ""

#: loader/net.c:508
#, c-format
msgid ""
"\n"
"This cannot be corrected in cmdline mode.\n"
"Halting.\n"
msgstr ""

#: loader/net.c:513 loader/net.c:568 pyanaconda/textw/netconfig_text.py:403
#, fuzzy
msgid "Network Error"
msgstr "Без паролю"

#: loader/net.c:599
msgid "Enable IPv4 support"
msgstr ""

#: loader/net.c:613
msgid "Enable IPv6 support"
msgstr ""

#: loader/net.c:653
msgid "Configure TCP/IP"
msgstr "Наладка TCP/IP"

#: loader/net.c:710
#, fuzzy
msgid "Missing Protocol"
msgstr "Адсутнічае група"

#: loader/net.c:711
#, fuzzy
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr "Вы мусіце вылучыць ня менш за адну мову для ўсталёўкі."

#: loader/net.c:718
msgid "IPv4 Needed for NFS"
msgstr ""

#: loader/net.c:719
msgid "NFS installation method requires IPv4 support."
msgstr ""

#: loader/net.c:827
#, fuzzy
msgid "IPv4 address:"
msgstr "АйПі адрас:"

#: loader/net.c:839 loader/net.c:906 data/ui/netconfig.glade.h:1
msgid "/"
msgstr ""

#: loader/net.c:894
#, fuzzy
msgid "IPv6 address:"
msgstr "АйПі адрас:"

#: loader/net.c:960 pyanaconda/textw/netconfig_text.py:184
#: pyanaconda/textw/netconfig_text.py:224
msgid "Gateway:"
msgstr "Брама:"

#: loader/net.c:968
#, fuzzy
msgid "Name Server:"
msgstr "DNS паслужнік"

#: loader/net.c:1007
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""

#: loader/net.c:1024
#, fuzzy
msgid "Manual TCP/IP Configuration"
msgstr "Наладка z/IPL"

#: loader/net.c:1148 loader/net.c:1156
msgid "Missing Information"
msgstr ""

#: loader/net.c:1149
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""

#: loader/net.c:1157
#, fuzzy
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr "Вы ўвялі нерэчаісны IP адрас."

#: loader/net.c:1655
#, c-format
msgid "Bad argument to kickstart network command: %s"
msgstr ""

#: loader/net.c:1676
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr ""

#: loader/net.c:1781
#, fuzzy
msgid "Seconds:"
msgstr "Другасны DNS:"

#: loader/net.c:1976
msgid "Networking Device"
msgstr "Сеткавая прылада"

#: loader/net.c:1977
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""

#: loader/net.c:1981
msgid "Identify"
msgstr ""

#: loader/net.c:1990
msgid "You can identify the physical port for"
msgstr ""

#: loader/net.c:1992
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""

#: loader/net.c:1999
msgid "Identify NIC"
msgstr ""

#: loader/net.c:2012
#, fuzzy
msgid "Invalid Duration"
msgstr "Нерэчаісныя зьвесткі"

#: loader/net.c:2013
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr ""

#: loader/net.c:2025
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr ""

#: loader/net.c:2195 loader/net.c:2199
#, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr ""

#: loader/nfsinstall.c:93
msgid "NFS server name:"
msgstr "Назва паслужніка NFS:"

#: loader/nfsinstall.c:97
#, c-format
msgid "%s directory:"
msgstr "%s тэчка:"

#: loader/nfsinstall.c:101
msgid "NFS mount options (optional):"
msgstr ""

#: loader/nfsinstall.c:107
#, fuzzy, c-format
msgid ""
"Please enter the server and path to your %s installation image and "
"optionally additional NFS mount options."
msgstr "Калі ласка, вылучыце манітор, які падключаны да сыстэмы."

#: loader/nfsinstall.c:115
msgid "NFS Setup"
msgstr "Наладка NFS"

#: loader/nfsinstall.c:310
msgid "That directory could not be mounted from the server."
msgstr ""

#: loader/nfsinstall.c:322
#, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr ""

#: loader/nfsinstall.c:415
#, c-format
msgid "Bad argument to NFS kickstart method command: %s"
msgstr ""

#: loader/urlinstall.c:138
#, fuzzy, c-format
msgid "Unable to retrieve %s."
msgstr "Немагчыма выцягнуць %s://%s/%s/%s."

#: loader/urlinstall.c:222
msgid "Unable to retrieve the install image."
msgstr ""

#: loader/urlinstall.c:375
#, c-format
msgid "Bad argument to URL kickstart method command: %s"
msgstr ""

#: loader/urlinstall.c:386
msgid "Must supply a --url argument to Url kickstart method."
msgstr ""

#: loader/urlinstall.c:393
#, c-format
msgid "Unknown Url method %s"
msgstr "Невядомы URL мэтад %s"

#: loader/urls.c:178 loader/urls.c:186
msgid "Retrieving"
msgstr "Выцягваньне"

#: loader/urls.c:272
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""

#: loader/urls.c:290
msgid "Enable HTTP proxy"
msgstr ""

#: loader/urls.c:304
#, fuzzy
msgid "Proxy URL"
msgstr "Нерэчаісная метка загрузкі"

#: loader/urls.c:309
#, fuzzy
msgid "Username"
msgstr "Уліковае імя"

#: loader/urls.c:314
msgid "Password"
msgstr "Пароль"

#: loader/urls.c:333
#, fuzzy
msgid "URL Setup"
msgstr "Наладка NFS"

#: loader/urls.c:341
#, fuzzy
msgid "You must enter a URL."
msgstr "Вы мусіце задаць назву тэчкі."

#: loader/urls.c:347
msgid "URL must be either an ftp or http URL"
msgstr ""

#: loader/urls.c:356
msgid "Proxy must be either an ftp or http URL"
msgstr ""

#: loader/windows.c:66
msgid "Loading SCSI driver"
msgstr "Загрузка кіроўцы SCSI"

#: loader/windows.c:67
#, fuzzy, c-format
msgid "Loading %s driver"
msgstr "Загрузка кіроўцы SCSI"

#: pyanaconda/storage/__init__.py:110
#, fuzzy
msgid "Unknown Device"
msgstr "Невядомая картка"

#: pyanaconda/storage/__init__.py:111
#, fuzzy, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""
"Узьнікла памылка чытаньня ўсталёўкі з ISO вобразаў. Калі ласка, праверце "
"вашыя ISO вобразы й паўтарыце спробу."

#: pyanaconda/storage/__init__.py:120
#, fuzzy
msgid "No disks found"
msgstr "Кіроўца ня знойдзены"

#: pyanaconda/storage/__init__.py:121
msgid "No usable disks have been found."
msgstr ""

#: pyanaconda/storage/__init__.py:132
msgid "Installation cannot continue."
msgstr ""

#: pyanaconda/storage/__init__.py:133
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""

#: pyanaconda/storage/__init__.py:161
#, fuzzy
msgid "Encrypt device?"
msgstr "Ключ шыфраваньня"

#: pyanaconda/storage/__init__.py:162
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""

#: pyanaconda/storage/__init__.py:188 pyanaconda/storage/devicetree.py:99
#, fuzzy
msgid "Confirm"
msgstr "Па_цьверджаньне:"

#: pyanaconda/storage/__init__.py:189
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""

#: pyanaconda/storage/__init__.py:194
#, fuzzy
msgid "Go _Back"
msgstr "На_зад"

#: pyanaconda/storage/__init__.py:195
#, fuzzy
msgid "_Write Changes to Disk"
msgstr "Зьмяніць"

#: pyanaconda/storage/__init__.py:214
#, fuzzy
msgid "Running..."
msgstr "Далучэньне..."

#: pyanaconda/storage/__init__.py:215
msgid "Storing encryption keys"
msgstr ""

#: pyanaconda/storage/__init__.py:231
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr ""

#: pyanaconda/storage/__init__.py:377
#, fuzzy
msgid "Examining Devices"
msgstr "Адсутнічае пакет"

#: pyanaconda/storage/__init__.py:378
#, fuzzy
msgid "Examining storage devices"
msgstr "Адсутнічае пакет"

#: pyanaconda/storage/__init__.py:657
msgid "This partition is holding the data for the hard drive install."
msgstr ""

#: pyanaconda/storage/__init__.py:662
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr ""

#: pyanaconda/storage/__init__.py:668
#, python-format
msgid "This device is part of the RAID device %s."
msgstr ""

#: pyanaconda/storage/__init__.py:671
msgid "This device is part of a RAID device."
msgstr ""

#: pyanaconda/storage/__init__.py:676
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr ""

#: pyanaconda/storage/__init__.py:679
msgid "This device is part of a LVM volume group."
msgstr ""

#: pyanaconda/storage/__init__.py:695
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""

#: pyanaconda/storage/__init__.py:970
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: pyanaconda/storage/__init__.py:975
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""

#: pyanaconda/storage/__init__.py:981
#, python-format
msgid ""
"Your / partition is less than %(min)s MB which is lower than recommended for "
"a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage/__init__.py:991
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""

#: pyanaconda/storage/__init__.py:998
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage/__init__.py:1027
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""

#: pyanaconda/storage/__init__.py:1030
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""

#: pyanaconda/storage/__init__.py:1037
msgid ""
"You have not specified a swap partition.  Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""

#: pyanaconda/storage/__init__.py:1042
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage/__init__.py:1049
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage/__init__.py:1053
#, fuzzy, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "Выдліць усе падзелы з гэтае сыстэмы"

#: pyanaconda/storage/__init__.py:1064
msgid "No Drives Found"
msgstr ""

#: pyanaconda/storage/__init__.py:1065
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/storage/__init__.py:1363 pyanaconda/storage/__init__.py:1372
msgid "Dirty File Systems"
msgstr ""

#: pyanaconda/storage/__init__.py:1364
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""

#: pyanaconda/storage/__init__.py:1373
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: pyanaconda/storage/__init__.py:1850
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""

#: pyanaconda/storage/__init__.py:1861
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""

#: pyanaconda/storage/__init__.py:1869
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""

#: pyanaconda/storage/__init__.py:1881
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a supported swap volume.  In order to continue "
"installation, you will need to format the device or skip it."
msgstr ""

#: pyanaconda/storage/__init__.py:1892
#, fuzzy, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""
"Немагчыма задзейнічаць прастору падмему %s: %s\n"
"\n"
"Верагодна, яна не была праініцыялізавана.\n"
"\n"
"Націсьніце \"Добра\", каб перазагрузіць сыстэму."

#: pyanaconda/storage/__init__.py:1899
#, fuzzy, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"Немагчыма задзейнічаць прастору падмему %s: %s\n"
"\n"
"Верагодна, яна не была праініцыялізавана.\n"
"\n"
"Націсьніце \"Добра\", каб перазагрузіць сыстэму."

#: pyanaconda/storage/__init__.py:1962 pyanaconda/storage/__init__.py:1974
msgid "Invalid mount point"
msgstr "Нерэчаісны пункт мацаваньня"

#: pyanaconda/storage/__init__.py:1963
#, fuzzy, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Узьнікла памылка пад час ініцыялізаваньня прасторы падмену %s. Гэта "
"сур'ёзная праблема й усталёўка ня можа працягвацца далей.\n"
"\n"
"Націсніце <Enter> для перазагрузкі сыстэмы."

#: pyanaconda/storage/__init__.py:1975
#, fuzzy, python-format
msgid ""
"An error occurred when trying to create %(mountpoint)s: %(msg)s.  This is a "
"fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Узьнікла памылка пад час ініцыялізаваньня прасторы падмену %s. Гэта "
"сур'ёзная праблема й усталёўка ня можа працягвацца далей.\n"
"\n"
"Націсніце <Enter> для перазагрузкі сыстэмы."

#: pyanaconda/storage/__init__.py:1991 pyanaconda/storage/__init__.py:2015
#, fuzzy
msgid "Unable to mount filesystem"
msgstr "Немагчыма зрабіць загрузачны дыск"

#: pyanaconda/storage/__init__.py:1992
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s.  You may "
"continue installation, but there may be problems."
msgstr ""

#: pyanaconda/storage/__init__.py:2016
#, fuzzy, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s: %(msg)s. This "
"is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"Узьнікла памылка пад час ініцыялізаваньня прасторы падмену %s. Гэта "
"сур'ёзная праблема й усталёўка ня можа працягвацца далей.\n"
"\n"
"Націсніце <Enter> для перазагрузкі сыстэмы."

#: pyanaconda/storage/dasd.py:155
#, fuzzy
msgid "Formatting DASD Device"
msgid_plural "Formatting DASD Devices"
msgstr[0] "Фарматаваньне"
msgstr[1] "Фарматаваньне"

#: pyanaconda/storage/dasd.py:156
#, python-format
msgid "Preparing %d DASD device for use with Linux..."
msgid_plural "Preparing %d DASD devices for use with Linux..."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/storage/devicelibs/lvm.py:312
#, python-format
msgid "vginfo failed for %s"
msgstr ""

#: pyanaconda/storage/devicelibs/lvm.py:341
#, fuzzy, python-format
msgid "lvs failed for %s"
msgstr "Падрабязнасьці для \"%s\""

#: pyanaconda/storage/devices.py:1300 pyanaconda/storage/devices.py:1998
#: pyanaconda/storage/devices.py:2414 pyanaconda/storage/devices.py:2909
#: pyanaconda/storage/devices.py:3383
#, fuzzy
msgid "Creating"
msgstr "Фарматаваньне"

#: pyanaconda/storage/devices.py:1301 pyanaconda/storage/devices.py:1999
#: pyanaconda/storage/devices.py:2415 pyanaconda/storage/devices.py:2910
#, fuzzy, python-format
msgid "Creating device %s"
msgstr "Зьмяніць"

#: pyanaconda/storage/devices.py:2508
#, python-format
msgid "A RAID%d set requires at least %d member"
msgid_plural "A RAID%d set requires at least %d members"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/storage/devices.py:3384
#, fuzzy, python-format
msgid "Creating file %s"
msgstr "Праверка·на·пашкоджаныя·блёкі прылады /dev/%s..."

#: pyanaconda/storage/devicetree.py:100
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""

#: pyanaconda/storage/devicetree.py:1897
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr ""

#: pyanaconda/storage/fcoe.py:66 pyanaconda/storage/fcoe.py:67
msgid "Connecting to FCoE SAN"
msgstr ""

#: pyanaconda/storage/fcoe.py:111
#, fuzzy
msgid "FCoE not available"
msgstr "Сеткавыя прылады"

#: pyanaconda/storage/formats/fs.py:102
msgid "filesystem configuration missing a type"
msgstr ""

#: pyanaconda/storage/formats/fs.py:354 pyanaconda/storage/formats/luks.py:194
#: pyanaconda/storage/formats/lvmpv.py:103
#: pyanaconda/storage/formats/swap.py:161
msgid "Formatting"
msgstr "Фарматаваньне"

#: pyanaconda/storage/formats/fs.py:355
#, fuzzy, python-format
msgid "Creating %s filesystem on %s"
msgstr "Праверка·на·пашкоджаныя·блёкі прылады /dev/%s..."

#: pyanaconda/storage/formats/fs.py:463
#, fuzzy
msgid "Resizing"
msgstr "Выцягваньне"

#: pyanaconda/storage/formats/fs.py:464
#, fuzzy, python-format
msgid "Resizing filesystem on %s"
msgstr "Фарматаваньне файлавае сыстэмы %s..."

#: pyanaconda/storage/formats/fs.py:499
#, fuzzy, python-format
msgid "Unknown return code: %d."
msgstr "Невядомы URL мэтад %s"

#: pyanaconda/storage/formats/fs.py:513
#, fuzzy
msgid "Checking"
msgstr "Праверка \"%s\"..."

#: pyanaconda/storage/formats/fs.py:514
#, fuzzy, python-format
msgid "Checking filesystem on %s"
msgstr "Праверка·на·пашкоджаныя·блёкі прылады /dev/%s..."

#: pyanaconda/storage/formats/fs.py:531
#, python-format
msgid "%(type)s filesystem check failure on %(device)s: "
msgstr ""

#: pyanaconda/storage/formats/fs.py:537
msgid ""
"Errors like this usually mean there is a problem with the filesystem that "
"will require user interaction to repair.  Before restarting installation, "
"reboot to rescue mode or another system that allows you to repair the "
"filesystem interactively.  Restart installation after you have corrected the "
"problems on the filesystem."
msgstr ""

#: pyanaconda/storage/formats/fs.py:545 pyanaconda/storage/partitioning.py:282
msgid "Unrecoverable Error"
msgstr "Невыправімая памылка"

#: pyanaconda/storage/formats/fs.py:889
msgid "File system errors left uncorrected."
msgstr ""

#: pyanaconda/storage/formats/fs.py:890
#, fuzzy
msgid "Operational error."
msgstr "Памылка"

#: pyanaconda/storage/formats/fs.py:891
msgid "Usage or syntax error."
msgstr ""

#: pyanaconda/storage/formats/fs.py:892
msgid "e2fsck cancelled by user request."
msgstr ""

#: pyanaconda/storage/formats/fs.py:893
msgid "Shared library error."
msgstr ""

#: pyanaconda/storage/formats/fs.py:1052
msgid ""
"Recoverable errors have been detected or dosfsck has discovered an internal "
"inconsistency."
msgstr ""

#: pyanaconda/storage/formats/fs.py:1054
msgid "Usage error."
msgstr ""

#: pyanaconda/storage/formats/luks.py:48
#, fuzzy
msgid "Encrypted"
msgstr "Ключ шыфраваньня"

#: pyanaconda/storage/formats/luks.py:195
#, fuzzy, python-format
msgid "Encrypting %s"
msgstr "Ключ шыфраваньня"

#: pyanaconda/storage/formats/lvmpv.py:104
#: pyanaconda/storage/formats/swap.py:162
#, fuzzy, python-format
msgid "Creating %s on %s"
msgstr "Праверка·на·пашкоджаныя·блёкі прылады /dev/%s..."

#: pyanaconda/storage/iscsi.py:83 pyanaconda/storage/iscsi.py:84
#, fuzzy
msgid "Scanning iSCSI nodes"
msgstr "Загрузка кіроўцы SCSI"

#: pyanaconda/storage/iscsi.py:135
msgid "Unable to change iSCSI initiator name once set"
msgstr ""

#: pyanaconda/storage/iscsi.py:137
#, fuzzy
msgid "Must provide an iSCSI initiator name"
msgstr "Вы мусіце задаць назву вузла."

#: pyanaconda/storage/iscsi.py:177 pyanaconda/storage/iscsi.py:178
msgid "Initializing iSCSI initiator"
msgstr ""

#: pyanaconda/storage/iscsi.py:214
msgid "iSCSI not available"
msgstr ""

#: pyanaconda/storage/iscsi.py:216
#, fuzzy
msgid "No initiator name set"
msgstr "Нерэчаісныя тэчкі"

#: pyanaconda/storage/iscsi.py:230
msgid "No iSCSI nodes discovered"
msgstr ""

#: pyanaconda/storage/iscsi.py:233 pyanaconda/storage/iscsi.py:234
msgid "Logging in to iSCSI nodes"
msgstr ""

#: pyanaconda/storage/iscsi.py:260
msgid "No new iSCSI nodes discovered"
msgstr ""

#: pyanaconda/storage/iscsi.py:263
msgid "Could not log in to any of the discovered nodes"
msgstr ""

#: pyanaconda/storage/partitioning.py:197
msgid ""
"Could not find enough free space for automatic partitioning.  Press 'OK' to "
"exit the installer."
msgstr ""

#: pyanaconda/storage/partitioning.py:200
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""

#: pyanaconda/storage/partitioning.py:229
msgid "Warnings During Automatic Partitioning"
msgstr "Папярэджаньні пад час аўтаматычнага разбіеньня"

#: pyanaconda/storage/partitioning.py:231
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"Пад час аўтаматычнага разбіеньня адбыліся наступныя папярэджаньні:\n"
"\n"
"%s"

#: pyanaconda/storage/partitioning.py:245
#: pyanaconda/storage/partitioning.py:266
#, fuzzy
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"Націсьніце \"Добра\", каб перазагрузіць сыстэму."

#: pyanaconda/storage/partitioning.py:247
#, fuzzy, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%(msg)s.%(extra)s"
msgstr ""
"Немагчыма разьмеркаваць запатрабаваныя падзелы: \n"
"\n"
"%s.%s"

#: pyanaconda/storage/partitioning.py:268
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""

#: pyanaconda/storage/partitioning.py:270
msgid "Automatic Partitioning Errors"
msgstr "Памылкі пад час аўтаматычнага разбіеньня"

#: pyanaconda/storage/partitioning.py:271
#, fuzzy, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %(extra)s"
msgstr ""
"Пад час стварэньня падзелаў адбыліся наступныя памылкі:\n"
"\n"
"%s\n"
"\n"
"Гэта магло здарыцца, калі бракуе вольнае прасторы для ўсталёўкі на вашым "
"жорсткім дыске ці дысках.%s"

#: pyanaconda/storage/zfcp.py:51
msgid "You have not specified a device number or the number is invalid"
msgstr ""

#: pyanaconda/storage/zfcp.py:53
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr ""

#: pyanaconda/storage/zfcp.py:55
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr ""

#: pyanaconda/storage/zfcp.py:133
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr ""

#: pyanaconda/storage/zfcp.py:143
#, python-format
msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:154
#, python-format
msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:161
#, python-format
msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
msgstr ""

#: pyanaconda/storage/zfcp.py:176
#, python-format
msgid ""
"Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:182
#, python-format
msgid ""
"LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already configured."
msgstr ""

#: pyanaconda/storage/zfcp.py:194
#, python-format
msgid ""
"Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on zFCP "
"device %(devnum)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:203
#, python-format
msgid ""
"Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
"again."
msgstr ""

#: pyanaconda/storage/zfcp.py:260
#, python-format
msgid ""
"Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
"%(fcplun)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:269
#, python-format
msgid ""
"Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:287
#, python-format
msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:313
#, python-format
msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
msgstr ""

#: pyanaconda/textw/complete_text.py:32
msgid "Reboot"
msgstr ""

#: pyanaconda/textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr "<Enter> для выхаду"

#: pyanaconda/textw/complete_text.py:62
msgid "Complete"
msgstr ""

#: pyanaconda/textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "Вылучэньне клявіятуры"

#: pyanaconda/textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr "Якая мадэля клявіятуры далучана да сыстэмы?"

#: pyanaconda/textw/netconfig_text.py:43
#, fuzzy, python-format
msgid "A value is required for the field %s"
msgstr "Значэньне неабходня для поля \"%s\"."

#: pyanaconda/textw/netconfig_text.py:126
#: pyanaconda/textw/netconfig_text.py:152 data/ui/netconfig.glade.h:9
msgid "Enable network interface"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:129
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please select network interface to configure."
msgstr ""

#: pyanaconda/textw/netconfig_text.py:155
#, fuzzy, python-format
msgid "Configure interface %s to be used during installation process."
msgstr "На якой мовы вы жадаеце весьці ўсталёўку?"

#: pyanaconda/textw/netconfig_text.py:160
msgid "Enable IPv4 Support"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:167
#: pyanaconda/textw/netconfig_text.py:207
#, fuzzy
msgid "Manual Configuration"
msgstr "Наладка z/IPL"

#: pyanaconda/textw/netconfig_text.py:175
#, fuzzy
msgid "IPv4 Address:"
msgstr "АйПі адрас:"

#: pyanaconda/textw/netconfig_text.py:189
#: pyanaconda/textw/netconfig_text.py:229
#, fuzzy
msgid "Nameserver:"
msgstr "DNS паслужнік"

#: pyanaconda/textw/netconfig_text.py:197
msgid "Enable IPv6 Support"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:204
#, fuzzy
msgid "Dynamic IP Configuration (DHCPv6)"
msgstr "Наладка SILO"

#: pyanaconda/textw/netconfig_text.py:215
#, fuzzy
msgid "IPv6 Address:"
msgstr "АйПі адрас:"

#: pyanaconda/textw/netconfig_text.py:270
#, fuzzy
msgid "Missing protocol"
msgstr "Адсутнічае група"

#: pyanaconda/textw/netconfig_text.py:271
#, fuzzy
msgid "You must select at least one protocol version"
msgstr "Вы мусіце вылучыць ня менш за адну мову для ўсталёўкі."

#: pyanaconda/textw/netconfig_text.py:279
#: pyanaconda/textw/netconfig_text.py:282
#, fuzzy
msgid "IPv4 Address"
msgstr "АйПі адрас:"

#: pyanaconda/textw/netconfig_text.py:286
#: pyanaconda/textw/netconfig_text.py:291
#, fuzzy
msgid "IPv4 Prefix"
msgstr "Нерэчаісны радок IP адраса"

#: pyanaconda/textw/netconfig_text.py:298
#, fuzzy
msgid "IPv4 Gateway"
msgstr "Брама"

#: pyanaconda/textw/netconfig_text.py:307
#, fuzzy
msgid "IPv4 Nameserver"
msgstr "DNS паслужнік"

#: pyanaconda/textw/netconfig_text.py:315
#: pyanaconda/textw/netconfig_text.py:318
#, fuzzy
msgid "IPv6 Address"
msgstr "АйПі Адрас"

#: pyanaconda/textw/netconfig_text.py:322
#: pyanaconda/textw/netconfig_text.py:327
#, fuzzy
msgid "IPv6 Prefix"
msgstr "Нерэчаісны радок IP адраса"

#: pyanaconda/textw/netconfig_text.py:326
#, fuzzy
msgid "IPv6 CIDR prefix must be between 0 and 128."
msgstr "IP адрас мусіць утрымліваць ліжбы ў прамежку ад 1 да 255"

#: pyanaconda/textw/netconfig_text.py:334
#, fuzzy
msgid "IPv6 Gateway"
msgstr "Брама"

#: pyanaconda/textw/netconfig_text.py:342
#, fuzzy
msgid "IPv6 Nameserver"
msgstr "DNS паслужнік"

#: pyanaconda/textw/netconfig_text.py:399
msgid "Configuring Network Interfaces"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:404
#, fuzzy, python-format
msgid "There was an error configuring network device %s"
msgstr "Наладка сеткі для %s"

#: pyanaconda/textw/partition_text.py:59
#, fuzzy
msgid "Partitioning Type"
msgstr "Памылка стварэньня падзелаў"

#: pyanaconda/textw/partition_text.py:61
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""

#: pyanaconda/textw/partition_text.py:64
#, fuzzy
msgid "Use entire drive"
msgstr "Выкарыстоўваць тэкставы рэжым"

#: pyanaconda/textw/partition_text.py:65
msgid "Replace existing Linux system"
msgstr ""

#: pyanaconda/textw/partition_text.py:66
#, fuzzy
msgid "Use free space"
msgstr "Бракуе вольнае прасторы"

#: pyanaconda/textw/partition_text.py:81
msgid "Which drive(s) do you want to use for this installation?"
msgstr ""

#: pyanaconda/textw/partition_text.py:96
#, fuzzy
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr ""
"  <Tab>/<Alt-Tab> паміж элемэнтамі | <Прагал> вылучыць | <F12> наступны "
"экран "

#: pyanaconda/textw/progress_text.py:46
msgid "Package Installation"
msgstr "Усталёўваньне пакета"

#: pyanaconda/textw/timezone_text.py:74
#, fuzzy
msgid "In which time zone are you located?"
msgstr "У якім часавым поясе вы знаходзіцеся?"

#: pyanaconda/textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr ""

#: pyanaconda/textw/upgrade_bootloader_text.py:111
#: pyanaconda/textw/upgrade_bootloader_text.py:121
msgid "Update boot loader configuration"
msgstr "Абнавіць наладку загрузчыка"

#: pyanaconda/textw/upgrade_bootloader_text.py:124
msgid "Skip boot loader updating"
msgstr "Абмінуць абнаўленьне наладкі загрузчыка"

#: pyanaconda/textw/upgrade_text.py:125
msgid "Free Space"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:143
msgid "RAM detected (MB):"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:146
msgid "Suggested size (MB):"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:149
msgid "Swap file size (MB):"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:157
msgid "Add Swap"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:182
msgid "The value you entered is not a valid number."
msgstr ""

#: pyanaconda/textw/upgrade_text.py:214
msgid "Reinstall System"
msgstr "Пераўсталёўка сыстэмы"

#: pyanaconda/textw/upgrade_text.py:226
msgid "System to Upgrade"
msgstr "Абнаўленьне сыстэмы"

#: pyanaconda/textw/upgrade_text.py:227
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""

#: pyanaconda/textw/userauth_text.py:30
msgid "Root Password"
msgstr "Пароль карыстальніка root"

#: pyanaconda/textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""

#: pyanaconda/textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr "Пароль карыстальніка "

#: pyanaconda/textw/userauth_text.py:84
#, fuzzy, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr "Вы насамрэч жадаеце зрабіць абнаўленьне?"

#: pyanaconda/textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "%s"

#: pyanaconda/textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"Вітаем на %s!\n"
"\n"

#: pyanaconda/textw/zipl_text.py:36
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""

#: pyanaconda/textw/zipl_text.py:68
msgid "z/IPL Configuration"
msgstr "Наладка z/IPL"

#: pyanaconda/textw/zipl_text.py:76 pyanaconda/textw/zipl_text.py:80
msgid "Chandev line "
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:1
#, fuzzy
msgid "Install"
msgstr "_Усталяваць %s"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:3
#, fuzzy
msgid "Install to Hard Drive"
msgstr "Пачатак усталёўкі"

#: liveinst:5
#, fuzzy
msgid "Starting Install to Hard Drive"
msgstr "Пачатак усталёўкі"

#: data/ui/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""

#: data/ui/GroupSelector.glade.h:2
#, fuzzy
msgid "_Deselect"
msgstr "В_ыдаліць"

#: data/ui/GroupSelector.glade.h:3
#, fuzzy
msgid "_Deselect all optional packages"
msgstr "Індывідуальна вы_лучыць пакеты"

#: data/ui/GroupSelector.glade.h:4
#, fuzzy
msgid "_Optional packages"
msgstr "Носьбіт якога тыпу ўтрымлівае пакеты для ўсталёўваньня?"

#: data/ui/GroupSelector.glade.h:5
#, fuzzy
msgid "_Select"
msgstr "Вылучыць _усё"

#: data/ui/GroupSelector.glade.h:6
#, fuzzy
msgid "_Select all optional packages"
msgstr "Індывідуальна вы_лучыць пакеты"

#: data/ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr ""

#: data/ui/account.glade.h:1
msgid "Confirm:"
msgstr ""

#: data/ui/account.glade.h:2
#, fuzzy
msgid "Root Password:"
msgstr "Пароль карыстальніка root"

#: data/ui/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"Рахунак root выкарыстоўваецца для адміністраваньня сыстэмы. Увядзіце "
"парольдля карыстальніка root."

#: data/ui/adddrive.glade.h:1
msgid "Add _FCoE SAN"
msgstr ""

#: data/ui/adddrive.glade.h:2
msgid "Add _ZFCP LUN"
msgstr ""

#: data/ui/adddrive.glade.h:3
msgid "Add _iSCSI target"
msgstr ""

#: data/ui/adddrive.glade.h:4
msgid "Advanced Storage Options"
msgstr ""

#: data/ui/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr ""

#: data/ui/adddrive.glade.h:6
#, fuzzy
msgid "_Add drive"
msgstr "Жорсткі дыск"

#: data/ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr ""

#: data/ui/addrepo.glade.h:2
#, fuzzy
msgid "<b>Repository _type:</b>"
msgstr "DNS паслужнік"

#: data/ui/addrepo.glade.h:4
#, fuzzy
msgid "Configure _proxy"
msgstr "Наладка %s"

#: data/ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""

#: data/ui/addrepo.glade.h:9
msgid ""
"Please provide the configuration information for this software repository."
msgstr ""

#: data/ui/addrepo.glade.h:10
#, fuzzy
msgid "Proxy U_RL (host:port)"
msgstr "Нерэчаісная метка загрузкі"

#: data/ui/addrepo.glade.h:11
#, fuzzy
msgid "Proxy pass_word"
msgstr "Без паролю"

#: data/ui/addrepo.glade.h:12
#, fuzzy
msgid "Proxy u_sername"
msgstr "Назва FTP паслужніка"

#: data/ui/addrepo.glade.h:13
#, fuzzy
msgid "Repository _URL"
msgstr "Нерэчаісная метка загрузкі"

#: data/ui/addrepo.glade.h:14
#, fuzzy
msgid "Select A Directory"
msgstr "Вылучыць усё"

#: data/ui/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr ""

#: data/ui/addrepo.glade.h:16
#, fuzzy
msgid "_Directory"
msgstr "%s тэчка:"

#: data/ui/addrepo.glade.h:17
#, fuzzy
msgid "_Options"
msgstr "Носьбіт якога тыпу ўтрымлівае пакеты для ўсталёўваньня?"

#: data/ui/addrepo.glade.h:18
#, fuzzy
msgid "_Partition"
msgstr "Дадаць падзел"

#: data/ui/addrepo.glade.h:19
msgid "_Path"
msgstr ""

#: data/ui/addrepo.glade.h:20
msgid "_Server"
msgstr "Пас_лужнік"

#: data/ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr ""

#: data/ui/anaconda.glade.h:4
msgid "_Next"
msgstr "На_перад"

#: data/ui/autopart.glade.h:1
#, fuzzy
msgid "<b>Shrink partition _to size (in MB):</b>"
msgstr "DNS паслужнік"

#: data/ui/autopart.glade.h:2
msgid "Re_view and modify partitioning layout"
msgstr ""

#: data/ui/autopart.glade.h:3
msgid "Volume to Shrink"
msgstr ""

#: data/ui/autopart.glade.h:4
#, fuzzy
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr "Які падзел зьяўляецца каранёвым для вашае сыстэмы?"

#: data/ui/autopart.glade.h:5
msgid "Which type of installation would you like?"
msgstr ""

#: data/ui/autopart.glade.h:6
#, fuzzy
msgid "_Encrypt system"
msgstr "Ключ шыфраваньня"

#: data/ui/autopart.glade.h:7
msgid "_Shrink"
msgstr ""

#: data/ui/blwhere.glade.h:1
#, fuzzy
msgid "/boot"
msgstr "Перазагруз_іць"

#: data/ui/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr ""

#: data/ui/blwhere.glade.h:3
#, fuzzy
msgid "Boot loader device"
msgstr "Загрузчык"

#: data/ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:6
#, fuzzy
msgid "MBR"
msgstr "SMB"

#: data/ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:9
#, fuzzy
msgid "Where would you like to install the boot loader for your system?"
msgstr "Вы насамрэч жадаеце зрабіць абнаўленьне?"

#: data/ui/cleardisks.glade.h:1
msgid "<b>Data Storage Devices</b> (to be mounted only)"
msgstr ""

#: data/ui/cleardisks.glade.h:2
#, fuzzy
msgid "<b>Install Target Devices</b>"
msgstr "_Усталяваць %s"

#: data/ui/cleardisks.glade.h:3
msgid ""
"<b>Tip:</b> Install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: data/ui/cleardisks.glade.h:4
msgid ""
"Below are the storage devices you've selected to be a part of this "
"installation.  Please indicate using the arrows below which devices you'd "
"like to use as data devices (these will not be formatted, only mounted) and "
"which devices you'd like to use as installation target devices (these may be "
"formatted).  Please also indicate which system drive will have the "
"bootloader installed."
msgstr ""

#: data/ui/create-storage.glade.h:1
msgid ""
"<span size=\"small\" color=\"gray\">Create a RAID formatted partition</span>"
msgstr ""

#: data/ui/create-storage.glade.h:2
msgid ""
"<span size=\"small\" color=\"gray\">Create a logical volume on selected "
"volume group</span>"
msgstr ""

#: data/ui/create-storage.glade.h:3
msgid ""
"<span size=\"small\" color=\"gray\">Create an LVM formatted partition</span>"
msgstr ""

#: data/ui/create-storage.glade.h:4
msgid ""
"<span size=\"small\" color=\"gray\">General purpose partition creation</span>"
msgstr ""

#: data/ui/create-storage.glade.h:5
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 1 free LVM formatted "
"partition</span>"
msgstr ""

#: data/ui/create-storage.glade.h:6
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 2 free RAID formatted "
"partitions</span>"
msgstr ""

#: data/ui/create-storage.glade.h:7
msgid "C_reate"
msgstr ""

#: data/ui/create-storage.glade.h:8
msgid "Create LVM"
msgstr ""

#: data/ui/create-storage.glade.h:9
#, fuzzy
msgid "Create Partition"
msgstr "Вылучыце падзел"

#: data/ui/create-storage.glade.h:10
msgid "Create Software RAID"
msgstr ""

#: data/ui/create-storage.glade.h:11
msgid "Create Storage"
msgstr ""

#: data/ui/create-storage.glade.h:12
msgid "LVM Logical Volume"
msgstr ""

#: data/ui/create-storage.glade.h:13
msgid "LVM Physical Volume"
msgstr ""

#: data/ui/create-storage.glade.h:14
msgid "LVM Volume Group"
msgstr ""

#: data/ui/create-storage.glade.h:16
#, fuzzy
msgid "RAID Partition"
msgstr "Дадаць падзел"

#: data/ui/create-storage.glade.h:17
#, fuzzy
msgid "Standard Partition"
msgstr "Дадаць падзел"

#: data/ui/detailed-dialog.glade.h:1
#, fuzzy
msgid "Info"
msgstr "Ігнараваць"

#: data/ui/detailed-dialog.glade.h:2
#, fuzzy
msgid "_Details"
msgstr "Падрабязнасьці"

#: data/ui/fcoe-config.glade.h:1
#, fuzzy
msgid "Configure FCoE Parameters"
msgstr "Парамэтры Chandev"

#: data/ui/fcoe-config.glade.h:2
msgid "NIC:"
msgstr ""

#: data/ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""

#: data/ui/fcoe-config.glade.h:5
msgid "Use DCB"
msgstr ""

#: data/ui/fcoe-config.glade.h:6
msgid "_Add FCoE Disk(s)"
msgstr ""

#: data/ui/filter.glade.h:1
msgid ""
"<b>Tip:</b> Selected drives will not be formatted unless you choose to do so "
"later during installation.  Also, note that post-installation you may mount "
"drives you did not select here by modifying your /etc/fstab file."
msgstr ""

#: data/ui/filter.glade.h:2
msgid "Add Advanced Target"
msgstr ""

#: data/ui/filter.glade.h:3
#, fuzzy
msgid "Basic Devices"
msgstr "Прылады"

#: data/ui/filter.glade.h:4
msgid "Filter By:"
msgstr ""

#: data/ui/filter.glade.h:5
msgid "Firmware RAID"
msgstr ""

#: data/ui/filter.glade.h:6
msgid "Identifier:"
msgstr ""

#: data/ui/filter.glade.h:7
msgid ""
"Interconnect\n"
"Vendor\n"
"Identifier"
msgstr ""

#: data/ui/filter.glade.h:10
msgid "LUN:"
msgstr ""

#: data/ui/filter.glade.h:11
msgid "Multipath Devices"
msgstr ""

#: data/ui/filter.glade.h:12
#, fuzzy
msgid "Other SAN Devices"
msgstr "RAID прылады"

#: data/ui/filter.glade.h:13
msgid ""
"Please select the drives you'd like to install the operating system on, as "
"well as any drives you'd like to automatically mount on your system:"
msgstr ""

#: data/ui/filter.glade.h:14
msgid ""
"Port / Target / LUN\n"
"Target Identifier"
msgstr ""

#: data/ui/filter.glade.h:16
msgid "Port:"
msgstr ""

#: data/ui/filter.glade.h:17
msgid "Search"
msgstr ""

#: data/ui/filter.glade.h:18
msgid "Search By:"
msgstr ""

#: data/ui/filter.glade.h:19
msgid "Search Results:"
msgstr ""

#: data/ui/filter.glade.h:21
#, no-c-format
msgid "Selected devices: %s (%s) out of %s (%s)."
msgstr ""

#: data/ui/filter.glade.h:22
msgid "Show Identifiers that Include:"
msgstr ""

#: data/ui/filter.glade.h:23
msgid "Show Only Devices From:"
msgstr ""

#: data/ui/filter.glade.h:24
msgid "Show Only Devices Using:"
msgstr ""

#: data/ui/filter.glade.h:25
msgid "Target:"
msgstr ""

#: data/ui/iscsi-config.glade.h:1
#, fuzzy
msgid "<b>CHAP _Password:</b>"
msgstr "_Пароль:"

#: data/ui/iscsi-config.glade.h:2
#, fuzzy
msgid "<b>CHAP _Username:</b>"
msgstr "Уліковае імя"

#: data/ui/iscsi-config.glade.h:3
#, fuzzy
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr "_Пароль:"

#: data/ui/iscsi-config.glade.h:4
#, fuzzy
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr "Уліковае імя"

#: data/ui/iscsi-config.glade.h:5
msgid "<b>_Target IP Address:</b>"
msgstr ""

#: data/ui/iscsi-config.glade.h:6
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr ""

#: data/ui/iscsi-config.glade.h:7
#, fuzzy
msgid "Configure iSCSI Parameters"
msgstr "Парамэтры Chandev"

#: data/ui/iscsi-config.glade.h:8
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: data/ui/iscsi-config.glade.h:9
msgid "_Add target"
msgstr ""

#: data/ui/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""

#: data/ui/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""

#: data/ui/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr ""

#: data/ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr ""

#: data/ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr ""

#: data/ui/netconfig.glade.h:2
#, fuzzy
msgid "<b>Gateway:</b>"
msgstr "Брама:"

#: data/ui/netconfig.glade.h:3
#, fuzzy
msgid "<b>IPv4 Address:</b>"
msgstr "АйПі Адрас"

#: data/ui/netconfig.glade.h:4
#, fuzzy
msgid "<b>IPv6 Address:</b>"
msgstr "АйПі Адрас"

#: data/ui/netconfig.glade.h:5
#, fuzzy
msgid "<b>Nameserver:</b>"
msgstr "DNS паслужнік"

#: data/ui/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr ""

#: data/ui/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr ""

#: data/ui/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr ""

#: data/ui/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr ""

#: data/ui/network.glade.h:1
#, fuzzy
msgid "Configure Network"
msgstr "Наладка сеткі для %s"

#: data/ui/network.glade.h:2
#, fuzzy
msgid "Hostname:"
msgstr "Назва вузла"

#: data/ui/network.glade.h:3
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""

#: data/ui/tasksel.glade.h:1
msgid "Customize _later"
msgstr ""

#: data/ui/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""

#: data/ui/tasksel.glade.h:4
#, no-c-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""

#: data/ui/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""

#: data/ui/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr ""

#: data/ui/tasksel.glade.h:7
msgid "_Customize now"
msgstr ""

#: data/ui/tasksel.glade.h:8
#, fuzzy
msgid "_Modify repository"
msgstr "Зь_мяніць падзел"

#: data/ui/zfcp-config.glade.h:1
#, fuzzy
msgid "<b>Device number:</b>"
msgstr "Уліковае імя"

#: data/ui/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr ""

#: data/ui/zfcp-config.glade.h:3
#, fuzzy
msgid "<b>WWPN:</b>"
msgstr "_Пароль:"

#: data/ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr ""

#: data/ui/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""

#: data/lang-table.h:1
msgid "Afrikaans"
msgstr ""

#: data/lang-table.h:2
msgid "Arabic"
msgstr "Арабская"

#: data/lang-table.h:3
msgid "Assamese"
msgstr ""

#: data/lang-table.h:4
msgid "Bengali"
msgstr "Бэнгальская"

#: data/lang-table.h:5
msgid "Bengali(India)"
msgstr "Бэнгальская (Індыя)"

#: data/lang-table.h:6
msgid "Bulgarian"
msgstr "Баўгарская"

#: data/lang-table.h:7
msgid "Catalan"
msgstr "Каталёнская"

#: data/lang-table.h:8
msgid "Chinese(Simplified)"
msgstr "Кітайская (спрошчаная)"

#: data/lang-table.h:9
msgid "Chinese(Traditional)"
msgstr "Кітайская (традыцыйная)"

#: data/lang-table.h:10
msgid "Croatian"
msgstr "Харвацкая"

#: data/lang-table.h:11
msgid "Czech"
msgstr "Чэская"

#: data/lang-table.h:12
msgid "Danish"
msgstr "Дацкая"

#: data/lang-table.h:13
msgid "Dutch"
msgstr "Галандцкая"

#: data/lang-table.h:14
msgid "English"
msgstr "Ангельская"

#: data/lang-table.h:15
msgid "Estonian"
msgstr "Эстонская"

#: data/lang-table.h:16
msgid "Finnish"
msgstr "Фінская"

#: data/lang-table.h:17
msgid "French"
msgstr "Француская"

#: data/lang-table.h:18
msgid "German"
msgstr "Нямецкая"

#: data/lang-table.h:19
msgid "Greek"
msgstr ""

#: data/lang-table.h:20
msgid "Gujarati"
msgstr "Гуяраці"

#: data/lang-table.h:21
msgid "Hebrew"
msgstr ""

#: data/lang-table.h:22
msgid "Hindi"
msgstr "Хіндзі"

#: data/lang-table.h:23
msgid "Hungarian"
msgstr "Вугорская"

#: data/lang-table.h:24
msgid "Icelandic"
msgstr "Іслянцкая"

#: data/lang-table.h:25
msgid "Iloko"
msgstr ""

#: data/lang-table.h:26
#, fuzzy
msgid "Indonesian"
msgstr "Макэдонская"

#: data/lang-table.h:27
msgid "Italian"
msgstr "Італійская"

#: data/lang-table.h:28
msgid "Japanese"
msgstr "Японская"

#: data/lang-table.h:29
msgid "Kannada"
msgstr ""

#: data/lang-table.h:30
msgid "Korean"
msgstr "Карэйская"

#: data/lang-table.h:31
msgid "Macedonian"
msgstr "Макэдонская"

#: data/lang-table.h:32
#, fuzzy
msgid "Maithili"
msgstr "Гуяраці"

#: data/lang-table.h:33
msgid "Malay"
msgstr "Малайская"

#: data/lang-table.h:34
#, fuzzy
msgid "Malayalam"
msgstr "Малайская"

#: data/lang-table.h:35
#, fuzzy
msgid "Marathi"
msgstr "Гуяраці"

#: data/lang-table.h:36
#, fuzzy
msgid "Nepali"
msgstr "Бэнгальская"

#: data/lang-table.h:37
#, fuzzy
msgid "Norwegian(Bokmål)"
msgstr "Нарвэская"

#: data/lang-table.h:38
msgid "Northern Sotho"
msgstr ""

#: data/lang-table.h:39
msgid "Oriya"
msgstr ""

#: data/lang-table.h:40
msgid "Persian"
msgstr "Пэрсыцкая"

#: data/lang-table.h:41
msgid "Polish"
msgstr "Польская"

#: data/lang-table.h:42
msgid "Portuguese"
msgstr "Партугальская"

#: data/lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr "Партугальская (Бразылія)"

#: data/lang-table.h:44
msgid "Punjabi"
msgstr "Пун'ябі"

#: data/lang-table.h:45
#, fuzzy
msgid "Romanian"
msgstr "Засталося"

#: data/lang-table.h:46
msgid "Russian"
msgstr "Расейская"

#: data/lang-table.h:47
#, fuzzy
msgid "Serbian"
msgstr "Пэрсыцкая"

#: data/lang-table.h:48
#, fuzzy
msgid "Serbian(Latin)"
msgstr "Пэрсыцкая"

#: data/lang-table.h:49
msgid "Sinhala"
msgstr ""

#: data/lang-table.h:50
#, fuzzy
msgid "Slovak"
msgstr "Славенская"

#: data/lang-table.h:51
msgid "Slovenian"
msgstr "Славенская"

#: data/lang-table.h:52
msgid "Spanish"
msgstr "Гішпанская"

#: data/lang-table.h:53
msgid "Swedish"
msgstr "Швэцкая"

#: data/lang-table.h:54
#, fuzzy
msgid "Tajik"
msgstr "Тамільская"

#: data/lang-table.h:55
msgid "Tamil"
msgstr "Тамільская"

#: data/lang-table.h:56
msgid "Telugu"
msgstr ""

#: data/lang-table.h:57
msgid "Turkish"
msgstr "Турэцкая"

#: data/lang-table.h:58
msgid "Ukrainian"
msgstr "Украінская"

#: data/lang-table.h:59
msgid "Vietnamese"
msgstr "Вьетнамская"

#: data/lang-table.h:60
msgid "Welsh"
msgstr "Валійская"

#: data/lang-table.h:61
msgid "Zulu"
msgstr "Зулу"

#~ msgid ""
#~ "You do not have enough RAM to use the graphical installer.  Starting text "
#~ "mode."
#~ msgstr ""
#~ "Вы ня маеце неабходнае колькасьці памяці, каб выкарыстоўваць "
#~ "графічнуюўсталёку. Запуск тэкставага рэжыму."

#, fuzzy
#~ msgid "Starting graphical installation."
#~ msgstr "Запуск графічнае ўсталёўкі..."

#~ msgid "Install class forcing text mode installation"
#~ msgstr "Пераход да ўсталёўкі ў тэкставым рэжыме"

#, fuzzy
#~ msgid "Installation Key"
#~ msgstr "Тып усталёўкі"

#, fuzzy
#~ msgid "The installer will now exit."
#~ msgstr "Усталёўка зараз скончыцца..."

#, fuzzy
#~ msgid ""
#~ "The following error was found while parsing thekickstart configuration "
#~ "file:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "Наступныя памылкі былі знойдзены цягам разбору наладкі kickstart:\n"
#~ "\n"
#~ "%s"

#, fuzzy
#~ msgid ""
#~ "An error was encountered while activating your storage configuration."
#~ msgstr ""
#~ "Наступныя памылкі былі знойдзены цягам разбору наладкі kickstart:\n"
#~ "\n"
#~ "%s"

#, fuzzy
#~ msgid "Invalid Key"
#~ msgstr "Нерэчаісны радок IP адраса"

#, fuzzy
#~ msgid "The key you entered is invalid."
#~ msgstr "Зьвесткі аб IP, якія вы ўвялі, зьяўляюцца нерэчаіснымі."

#, fuzzy
#~ msgid "_Skip"
#~ msgstr "Абмінуць"

#~ msgid "The installer will now exit..."
#~ msgstr "Усталёўка зараз скончыцца..."

#, fuzzy
#~ msgid "/boot is not on an ext2 filesystem."
#~ msgstr "Выдліць усе падзелы з гэтае сыстэмы"

#, fuzzy
#~ msgid "Please enter your %(instkey)s"
#~ msgstr "Калі ласка, абярыце тып уваходу:"

#~ msgid "%s not found"
#~ msgstr "%s ня знойдзена"

#~ msgid "%s %s installation"
#~ msgstr "%s %s усталёўка"

#, fuzzy
#~ msgid "Web server"
#~ msgstr "DNS паслужнік"

#, fuzzy
#~ msgid "Installation Number"
#~ msgstr "Тып усталёўкі"

#~ msgid "Unknown"
#~ msgstr "Невядома"

#, fuzzy
#~ msgid ""
#~ "The drive %s will now be cloned to the following drives:\n"
#~ "\n"
#~ msgstr ""
#~ "Назва вузла \"%s\" зьяўляецца нерэчаіснай з-за наступнае прычыны\n"
#~ "\n"
#~ "%s"

#~ msgid "Telnet"
#~ msgstr "Telnet"

#, fuzzy
#~ msgid "Running anaconda via telnet."
#~ msgstr "Анаконда працуе праз telnet..."

#~ msgid "Unknown Host"
#~ msgstr "Невядомы вузел"

#~ msgid "%s is not a valid hostname."
#~ msgstr "%s - нерэчаісная назва вузла.ы"

#, fuzzy
#~ msgid "Writing storage configuration to disk"
#~ msgstr "Абнавіць наладку загрузчыка"

#, fuzzy
#~ msgid "Missing Device"
#~ msgstr "Адсутнічае пакет"

#, fuzzy
#~ msgid "IPv4 Network Mask "
#~ msgstr "Адрас/маска"

#, fuzzy
#~ msgid "Add FCP Device"
#~ msgstr "FCP прылада"

#, fuzzy
#~ msgid "NIC device name"
#~ msgstr "Кіроўца ня знойдзены"

#, fuzzy
#~ msgid "%s is not a valid NIC device name."
#~ msgstr "%s - нерэчаісная назва вузла.ы"

#, fuzzy
#~ msgid "Target IP Address"
#~ msgstr "АйПі адрас:"

#, fuzzy
#~ msgid "iSCSI Initiator Name"
#~ msgstr "Нерэчаісныя тэчкі"

#, fuzzy
#~ msgid "CHAP username"
#~ msgstr "Назва FTP паслужніка"

#, fuzzy
#~ msgid "CHAP password"
#~ msgstr "Пароль"

#~ msgid "Create new boot loader configuration"
#~ msgstr "Стварыць новую наладку загрузчыка"

#, fuzzy
#~ msgid "_Advanced storage configuration"
#~ msgstr "Абнавіць наладку загрузчыка"

#, fuzzy
#~ msgid "What drive would you like to _boot this installation from?"
#~ msgstr "На якой мовы вы жадаеце весьці ўсталёўку?"

#, fuzzy
#~ msgid "Please enter your %(instkey)s."
#~ msgstr "Калі ласка, абярыце тып уваходу:"

#, fuzzy
#~ msgid "label1"
#~ msgstr "Метка"

#, fuzzy
#~ msgid ""
#~ "You do not have enough RAM to install %s on this machine.\n"
#~ "\n"
#~ "Press <return> to reboot your system.\n"
#~ msgstr "Бракуе памяці для ўсталёўкі %s на гэай машыне."

#, fuzzy
#~ msgid "Unable To File Bug"
#~ msgstr "Немагчыма рэдагаваць"

#, fuzzy
#~ msgid "Bug Filing Not Supported"
#~ msgstr "Не падтрымліваецца"

#, fuzzy
#~ msgid "Invalid Bug Information"
#~ msgstr "Нерэчаісныя зьвесткі аб IP"

#, fuzzy
#~ msgid "Unable To Login"
#~ msgstr "Немагчыма рэдагаваць"

#, fuzzy
#~ msgid "Bug Updated"
#~ msgstr "Абнаўленьні"

#~ msgid "Dump Written"
#~ msgstr "Дамп запісаны"

#, fuzzy
#~ msgid ""
#~ "Your system's state has been successfully written to the disk. The "
#~ "installer will now exit."
#~ msgstr ""
#~ "Стан сыстэмы быў пасьпяхова запісаны на гнуткі дыск. Зараз сыстэма будзе "
#~ "перазагружана."

#, fuzzy
#~ msgid "Dump Not Written"
#~ msgstr "Дамп запісаны"

#, fuzzy
#~ msgid ""
#~ "Your system's state has been successfully written to the remote host.  "
#~ "The installer will now exit."
#~ msgstr ""
#~ "Стан сыстэмы быў пасьпяхова запісаны на гнуткі дыск. Зараз сыстэма будзе "
#~ "перазагружана."

#~ msgid "Debug"
#~ msgstr "Адладка"

#~ msgid "Exception Occurred"
#~ msgstr "Здарылася выключэньне"

#~ msgid "None"
#~ msgstr "Няма"

#~ msgid "Save"
#~ msgstr "Захаваць"

#, fuzzy
#~ msgid "Save to local disk"
#~ msgstr "_Зрабіць загрузачны дыск"

#, fuzzy
#~ msgid "Bug Description"
#~ msgstr "Апісаньне: %s"

#~ msgid "%s Bytes"
#~ msgstr "%s байт"

#~ msgid "Processing"
#~ msgstr "Апрацоўка"

#, fuzzy
#~ msgid "boot flag not available for this partition"
#~ msgstr "Немагчыма разьмеркаваць падзелы"

#, fuzzy
#~ msgid "Bug _description"
#~ msgstr "Апісаньне: %s"

#, fuzzy
#~ msgid "_Password"
#~ msgstr "_Пароль:"
